---
title: "Parity and AD specific neurodegeneration"
author: "Clazz"
toc: true
number-sections: true
format: 
  html:
    code-fold: true
editor: source
bibliography: references.bib
csl: elsevier-vancouver.csl
---

```{r, include = F}

if (!require("pacman")) install.packages("pacman")

pacman::p_load(tidyverse, VIM, ggpubr, ggeffects, car, sjPlot,caret, ggrain, emmeans, skimr, lavaan, patchwork, lme4, lmerTest, gtsummary, huxtable, readxl, broom.mixed, pwrss)
```

```{r setup, include=FALSE}

# Project: ASHS_APOE_analysis
# Script: paper_ashs_pregnancy_APOE.R
# Author: Clazz
# Date: 25/09/2025
#
# DATA SOURCES:
# - ALFA+: G:/My Drive/Data/ALFA+
#
#


# Register an inline hook:
options(scipen = 1, digits = 3)

knitr::opts_chunk$set(
  fig.width = 8,
  fig.height = 8,
  out.width = "100%",
  dpi = 300
)

config <- config::get()

source("C:/Users/U253662/Documents/git/functions/jt_plot_aesthetics.R")
```


```{r, include=FALSE}
# creating basic dataset

apoe2<-read_delim("G:/My Drive/Data/ALFA+/data_raw/fluid_biomarkers/20201007_APlus_Biomarcadores - corregido.csv", 
    delim = ";", escape_double = FALSE, trim_ws = TRUE)%>%
  dplyr::select(IdParticipante, APOE_consensus)%>%
  rename(subject= IdParticipante)%>%
  mutate(APOE_consensus==as.character(APOE_consensus),
         APOE1 = str_sub(APOE_consensus, start = 1, end = 1),
         APOE2 = str_sub(APOE_consensus, start = 2, end = 2),
         APOE_cat = case_when(APOE1=="4" & APOE2 == "4" ~ "2",
                              APOE1== "4" & APOE2 != "4" ~ "1",
                              APOE1 != "4" & APOE2=="4" ~ "1",
                              .default = "0"))%>%
  dplyr::select(subject, APOE_cat)%>%
  unique()

data_naked<-read_delim(paste0(config$alfa_dataset_cleaned, "ALFA_dataset_basic.csv"), 
    delim = ",", escape_double = FALSE, trim_ws = TRUE)%>%
  mutate(sex=factor(sex),
         APOE_binary = factor(APOE_binary), 
         subject=as.numeric(subject),
         parous_group= factor(parous_group),
         DOB = as.Date (DOB, format = "%Y-%m-%d"),
         kids_dicho=factor(kids_dicho),
         kids_total2 = ifelse(kids_total==4, 3, kids_total))%>%
  dplyr::select(-TIV)%>%
  left_join(., apoe2)%>%
  mutate(kids_total = ifelse(kids_total ==5, 4, kids_total))
  

alfa_subjects<-data_naked%>%
  dplyr::select(subject, sex)

```


```{r hippocampus_data, include = F}
ashs1<-read_excel(paste0(config$alfa_dataset_neuroimaging, "ASHS/bx_ashs_volumes_ALFA_PLUS_V1_20231120_20250702_164509.xlsx"))%>%
  dplyr::select(subject, region, side, volume)%>%
  group_by(subject, region) %>%
  summarize(volume = sum(volume), .groups = "drop")%>%
  pivot_wider(names_from = region, values_from = volume, names_prefix = "vol_")%>%
  mutate(Visit = 1,
         diff_mri = 0,
         subject= as.numeric(subject))

ashs1b<-read_excel(paste0(config$alfa_dataset_neuroimaging, "ASHS/bx_ashs_volumes_ALFA_PLUS_EXP_20250918_161906.xlsx"))%>%
  dplyr::select(subject, region, side, volume)%>%
  group_by(subject, region) %>%
  summarize(volume = sum(volume), .groups = "drop")%>%
  pivot_wider(names_from = region, values_from = volume, names_prefix = "vol_")%>%
  mutate(Visit = 1,
         diff_mri = 0,
         subject= as.numeric(subject))%>%
  filter(subject != 11331,
         subject != 10634,
         subject!= 13368)

date_mri_v1<-read_excel(paste0(config$alfa_dataset_neuroimaging, "scan_dates/bx_scandates_ALFA_PLUS_V1_20231120_20250210_135308.xlsx"))%>%
  dplyr::select(subject_label, scandate)%>%
  mutate(subject_label = as.numeric(subject_label))%>%
  rename(subject = subject_label,
         scandate_v1 = scandate)

date_mri_v2<-read_excel(paste0(config$alfa_dataset_neuroimaging, "scan_dates/bx_scandates_ALFA_PLUS_V2_20230518_20250210_135351.xlsx"))%>%
  dplyr::select(subject_label, scandate)%>%
  mutate(subject_label = as.numeric(subject_label))%>%
  rename(subject = subject_label,
         scandate_v2 = scandate)

diff_mri<-full_join(date_mri_v1, date_mri_v2)%>%
  mutate(scandate_v1 = as.Date(scandate_v1, format = "%Y-%m-%d"),
         scandate_v2 = as.Date(scandate_v2, format = "%Y-%m-%d"),
         diff_mri = as.numeric(difftime(scandate_v2, scandate_v1, units = c("days")))/365.25)%>%
  dplyr::select(subject, diff_mri)
  

ashs2<-read_excel(paste0(config$alfa_dataset_neuroimaging, "ASHS/bx_ashs_volumes_ALFA_PLUS_V2_20230518_20250702_165801.xlsx"))%>%
  dplyr::select(subject, region, side, volume)%>%
  group_by(subject, region) %>%
  summarize(volume = sum(volume), .groups = "drop")%>%
  pivot_wider(names_from = region, values_from = volume, names_prefix = "vol_")%>%
  mutate(Visit = 2,
         subject= as.numeric(subject))%>%
  left_join(., diff_mri)


data_ashs_long<-ashs1%>%
  bind_rows(ashs1b)%>%
  bind_rows(ashs2)%>%
  mutate(subject = as.numeric(subject),
         vol_HIPP = vol_CA1+vol_CA2+vol_CA3+vol_DG+vol_SUB,
         across(starts_with("vol_"), ~./1000))%>%
  right_join(., data_naked)%>%
  filter(subject!= 21136, #this is a very big outlier that increases 1cm3 of HV over visits
         subject!= 12560, #this is a very big outlier that increases 1cm3 of HV over visits
         #subject!= 55854, #cooks distance > 1
         subject != 12186) #is missing CA2 and CA3 values in V1
         #subject != 11154 )#cooks distance > 1
 


# data_ashs_long_outliers<-ashs1%>%
#   bind_rows(ashs1b)%>%
#   bind_rows(ashs2)%>%
#   mutate(subject = as.numeric(subject),
#          vol_HIPP = vol_CA1+vol_CA2+vol_CA3+vol_DG+vol_SUB,
#          across(starts_with("vol_"), ~./1000))%>%
#   right_join(., data_naked)%>%
#   left_join(., A_long)

# data_ashs_long_outliers%>%
#   filter(sex==2)%>%
#   filter(subject== 21136|
#          subject== 12560)%>%
#   mutate(subject=factor(subject))%>%
#   ggplot(aes(x=diff_mri, y=vol_HIPP, colour=subject))+
#   geom_point()+
#   geom_line()+
#   th
# 
# data_ashs_long%>%
#   filter(sex==2)%>%
#   mutate(subject=factor(subject))%>%
#   ggplot(aes(x=Visit, y=vol_HIPP, group=subject))+
#   geom_point()+
#   geom_line()+
#   th
```

```{r, biomarkers, include=F}

biomarkers<-read_delim(paste0(config$alfa_dataset_cleaned, "biomarkers_CSF_processed.csv"), 
    delim = ",", escape_double = FALSE, trim_ws = TRUE)%>%
  mutate(A=factor(A))

biomarkers_time_diff<-read_delim(paste0(config$alfa_dataset_cleaned, "biomarkers_CSF_processed.csv"), 
    delim = ",", escape_double = FALSE, trim_ws = TRUE)%>%
  dplyr::select(subject, collection_dt, Visit)%>%
  pivot_wider(names_from = Visit, values_from = collection_dt)%>%
  mutate(V1 = as.Date(V1, format = "%d/%m/%Y"),
         V2 = as.Date(V2, format = "%d/%m/%Y"),
         biomarkers_time_diff = as.numeric(difftime(V2, V1, units = c("days")))/365.25)%>%
  dplyr::select(subject, biomarkers_time_diff)%>%
  mutate(Visit = 2)
  
data_biomarkers_long<-left_join(data_naked, biomarkers)%>%
  mutate(Visit=ifelse(Visit=="V1", 1, 2))%>%
  left_join(., biomarkers_time_diff)%>%
  mutate(biomarkers_time_diff = ifelse(Visit==1, 0, biomarkers_time_diff))

A_long<-data_biomarkers_long%>%
  dplyr::select(subject, Visit, CSF_Abeta_ratio)
  
```


```{r cognition_data, include=F}
cognition<-read_excel(paste0(config$alfa_dataset_cognition, "PACCv1_to_v3(incomplete).xlsx"))%>%
  dplyr::select(IdParticipante, Visit, Z_PACC, Z_FCSRTI_RT, Z_aWMS_DR_RS, Z_Wclave_totalscr, Z_FS_totalcnswr, FechaResultado)%>%
  rename(subject = IdParticipante)

time_diff<-read_excel(paste0(config$alfa_dataset_cognition, "PACCv1_to_v3(incomplete).xlsx"))%>%
  dplyr::select(IdParticipante, Visit,FechaResultado)%>%
  rename(subject = IdParticipante)%>%
  filter(Visit != 3)%>%
  pivot_wider(names_from = Visit, values_from = FechaResultado, names_prefix = c("Fecha_Resultado"))%>%
  mutate(time_diff = as.numeric(difftime(Fecha_Resultado2, Fecha_Resultado1, units= c("days"))))%>%
  dplyr::select(subject, time_diff)

data_cog_long<- left_join(data_naked, cognition)%>%
  mutate(age_cog = as.numeric(difftime(FechaResultado, DOB, units= c("days")))/365.25)%>%
  group_by(subject)%>%
  filter(Visit != 3) %>%
  filter(n()>=2)%>%
  relocate(Visit, .after=sex)%>%
  ungroup()%>%
  left_join(., time_diff)%>%
  mutate(time_diff = ifelse(Visit == 1, 0, time_diff),
         time_diff = time_diff/365.25)
```


# Abstract

300 words

# Introduction

Alzheimer's disease (AD) is a neurodegenerative disorder characterised by the accumulation of pathophysiological abnormalities leading to neurodegeneration and progressive cognitive and functional decline. Ageing, carring the apolipoprotein E E4 (APOE4) and having female chromosomal sex are three established non-modifiable risk factors for the typical late-onset of this disease @riedel2016. Carrying one copy of APOE4 increases AD risk by three to four fold, whilst carrying 2 copies further increases risk by 10 folds @corder1993. Women have a higher incidence of late-onset AD dementia @beam2018 and show faster cognitive decline after diagnosis of MCI or AD dementia @sexdiff2018. Amongst APOE4 carriers, women exhibit an increased risk of conversion from healthy ageing to MCI, or from MCI to AD dementia between the ages of 65 to 75 [@Altmann2014; @Neu2017]. Therefore, studying the interactive effects of these non-modifiable risk factors might help us better understand the reason behind the aforementioned sex differences in AD incidence and trajectory.

The hippocampus is one of the first regions affected by AD, and whole hippocampal volume is a hallmark predictor of conversion from MCI to AD @apostolova2006. APOE4 carriers show reduced HV compared with non carriers \[citation\].
Older EPOE4 carriers show reduced hippocampal volume compared with non-carriers, especially in the CA1 and subiculum tissue @pievani2011.

-   mention info on DG here?

Interestingly, this structure is deeply impacted by pregnancy in a lasting way @puri2023, but long-term effects remain vastly misunderstood. Animal models show decreased hippocampal neurogenesis and HV during gestation and postpartum period [@pawluski2007; @eid2019], but parous middle aged rats show increased hipocampal neurogenesis compared with nulliparous @lee2024. In humans, pregnancy is associated with overall gray matter volume loss, and hippocampal volume loss @Hoekzema2017. These changes are detectable up to 6 years post-partum in primiparous women @martínez-garcía2021. However, a study using 19,787 middle-aged CU women from the UK biobank found that middle-aged parous women exhibited reduced brain age compared with nulliparous women based on gray matter volume patterns @de2020. This effect was mainly driven by anatomical components situated in the limbic region, like the hippocampus and parahippocampal regions, putamen, amygdala and accumbens. This study also found significant positive linear associations between the Putamen, Hippocampus, Amygdala and Accumbens and number of childbirths.

-   repport effect sizes on the de Lange study to compare with yours?

Evidence on how previous pregnancies influence cognitive decline and AD risk is conflicting. Amongst epidemiological studies that explored a potential association between parity and AD, some report a link between multiparity and greater cognitive decline @colucci2006, whilst other report no link or the opposite findings [@Fox2018; @Fox2013.; @wolfova2024]Adittionally, a post-mortem histopathological study of 42 women, with clinical diagnosis spanning the spectrum from normal aging to definite AD, found that multiparity was associated with a greater number of neuritic plaques @Beeri2009a. So although parity seems to be assoicated with better neuroimaging outcomes, links with cognitive oucomes are mixed.

Lee and colleagues @lee2024 propose that this "parity paradox" could be reconciled using the Healthy cell bias hypothesis, where hormonal changes brought on by prgenancy could impact cell metabolism differently depending on presence of disease risk. Using wildtype and APOE4 models middle aged female rats, they show that APOE status modifies the effect of parity on neurogenesis and HV. Parity was associated with neurogenesis in wildtype rats, particularly in the ventral dentate gyrus, but decreased neurogenesis in APOE4 rats. In addition to this, parous APOE4 rats showed a spatial solving strategy which indicated a decreased recruitment of the hippocampus. To our knowledge, no study has explored this interaction of APOE4 and parity on hippocampal structure in humans.

Our objective was to explore potential links and mechanisms through which pregnancy interacts with AD genetc risk in CU postmenopausal women at risk of AD. We investigated the interplay between parity, APOE-E4 carriership, HV and... to test whether AD related genetic risk modifies the effect of parity on HV trajectory in middle aged women. We hypothesized that parous women would show decreased whole hippocampal volume loss but this relationship would be inversed in APOE4 carriers.



# Methods

## Study population

We included postmenopausal women who completed the baseline and first longitudinal visit (with an average time between visits of 3.32 years) from the ongoing ALFA+ study (Clinicaltrials.gov, NCT01835717). ALFA+ is a research cohort of middle-aged cognitively unimpaired participants, many of whom are offspring of AD patients, who have been deeply characterized by clinical interviews, lifestyle and risk factor questionnaires, cognitive testing, CSF biomarkers, and neuroimaging procedures, including magnetic resonance imaging (MRI), and Aβ positron emission tomography (PET). All of these procedures are repeated every 3 years with the main aim of identifying the earliest pathophysiological changes in the preclinical AD continuum @molinuevo2016. Briefly, ALFA+ inclusion criteria were: 1) participants who had previously participated in the 45-65/FPM2012 study (ALFA parent cohort); 2) age between 45 and 75 years at the moment of inclusion in the cohort; 3) long-term commitment to undergo all tests and study procedures (MRI, PET, and lumbar puncture). Exclusion criteria at baseline included: 1) cognitive impairment (Clinical Dementia Rating \[CDR\] \> 0, Mini-Mental State Examination \[MMSE\] \< 26, semantic fluency \< 12); 2) any significant systemic illness or unstable medical condition which could lead to difficulty complying with the protocol; 3) any contraindication to any test or procedure; 4) family history of monogenic AD. In addition, women with a history of cardiovascular-related gestational disease, such as preeclampsia, were excluded from our sample for analysis. Sex and menopausal status, defined as absence of menstruations for 12 consecutive months, were self disclosed. Women who reported being premenopausal or perimenopausal were not included in our analysis either.

The study was approved by the Independent Ethics Committee “Parc de Salut Mar”, Barcelona, and all participants gave written informed consent. Baseline visits took place between 2016 and 2019, and the follow-up visit between 2019 and 2022.

The sample consisted of 254 participants at baseline visit, and 211 at follow-up (Table 1). Out of the 43 participants who dropped out of the study between visits, 30 withdrew for personal reasons, 3 became unresponsive, 2 died, 5 showed incidental l findings on their structural MRI, and 3 were excluded due to mild cognitive impairment clinical diagnosis.

## Reproductive variables

Parity was defined as the number of biological children and used as a continuous and dichotomous [parous vs nulliparous] variable in the main analyses. Due to the small number of grand multiparous participants, anyone with 4 or more biological children was grouped together. Hormonal Replacement Therapy (HRT) use was defined as a binary variable [currently using vs not currently using]. Cause of menopause was also defined as a binary variable [natural vs non-natural]. Reproductive span was calculated by subtracting age at menarche from age at menopause. Early menopause was defined as a binary variable [age at menopause> 45 vs age at menopause≤ 45]. These informations were self-reported by participants during an interview with a clinician at the baseline visit. When asking about age at menopause, participants were asked to report their age 6(?) months after their last period.

## Cognitive measures

Cognition was defined using 5 composite scores that measured sparate congitive domains categorised as follows: attention, executive functions, memory, language and visual processing. Z-scores were computed for Visit 1 and 2 and used in the linear regressions. 

## AD biomarkers

CSF collection, processing, and storage in the ALFA+ study have been described previously @Milà-Alomà2020. Briefly, CSF Aβ42 and Aβ40 were measured with the exploratory Roche NeuroToolKit immunoassays (Roche Diagnostics International Ltd, Rotkreuz, Switzerland) on a cobas e 601 module. Measurements were performed at the Clinical Neurochemistry Laboratory, Sahlgrenska University Hospital, Molndal, Sweden. Aß accumulation (Aß+, Aß–) was estimated using CSF Aß42/40. Aß status (Aß+, Aß–) was defined using the cutoff of 0.071 for the ratio Aß42/40 [23], where any participant below this cutoff was considered to be Aβ+.

- add other inflammation biomarkers


## Imaging data acquisition and preprocessing

Automatic Segmentation of Hippocampal Subfields (ASHS) software @Yushkevich2015 was used on the T1, T2 and inversion recovery (IR) weighted images to segment the hippocampal formation in the following sub-regions: Brodmann areas 35 and 36, cornu Ammonis (CA) 1, 2 and 3, dentate gyrus (DG), entorhinal cortex (ERC), parahippocampal cortex (PHC), subiculum (SUB), and the hippocampal sulcus (SUL). All segmentations were visually inspected before proceeding with the statistical analyses.

Hippocampal values are reported in cm3 and were obtained at both baseline and follow-up visits.

Gray matter volume was obtained using Freesurfer7.

Total Intracranial volume (TIV) was estimated using deformation fields obtained during the nonlinear registration of each subject’s T1-weighted MRI to a whole-brain template. The FSL Brain Extraction Tool @smith2002 was applied to the template to create an intracranial mask, which was warped into each subject’s MRI at native space to obtain the TIV measurement.

## Statistical analyses

We used linear mixed-effects models to assess the joint effects of parity and APOE-e4 (exposures), on HV and cognition z-score (outcomes) over both visits, and test if this association is dependent on time. These models included a three-way interaction term among parity, APOE4, and time (the interval between Visit 1 and Visit 2 in years) and all lower-order interactions. Cognitive analyses were adjusted for years of education, and age at baseline visit. HV analyses were additionally adjusted for TIV. 

Analyses that showed a significant parity x APOE-e4 interaction were reproduced adding Aβ load as a covariate to disentangle the neuropathological contributions of Aβ accumulation from the phenogenetic contributions of APOE4. Given the high correlation between Aβ load and APOE4 variables, collinearity was tested for each model and can be found in the suppplementary materials. In addition to this, we reproduced the analysis in Aβ-negative participants only.

Specificity of the results was tested by testing the 3-way interaction between parity, APOE4 and time on gray matter volume changes in the calcarine cortex.

Sensitivity analyses were performed to explore the impact of additional reproductive variables on HV in the context of the 3-way interaction between parity APOE4 and time. These regressions were further adjusted by HRT, early menopause, reproductive span and cause of menopause.

Additional exploratory linear regressions were also performed to explore the interaction between parity and APOE-e4 positivity on hippocampal subfield volumes across visits. Subfields values are standardized. Given the exploratory nature of these analyses, results are reported without correction for multiple comparisons.

Multiple imputation was used on covariates with missing data in order to avoid imcomplete case deletion.

Statistical significance was set at p \< 0.05. Analyses were conducted using R software, version 4.5.1 (The R Foundation for Statistical Computing, Vienna, Austria). The package ‘emmeans’ was used to calculate estimated marginal mean for direct group comparison of predicted effects @lenth2017. The package tidiverse was used for data processing and plotting @wickham2019. The package gtsummary was used for formatting tables @sjoberg2021. The package ggeffects was used to extract predicted values from regression models @lüdecke2018. The package lme4 was used for performing mixed effect linear regression models @bates2015. The package MICE was used for multiple imputation of missing data.

Rates of missing data in exposures or outcome variables at each time point are reported (Table 1). 

- cognitive subfield analyses are given standardized 
- analysis with biomarker as outcome were normalised using box-cox regression


# Results

## Descriptive results

```{r, descriptives alfa, include = F}
alfa_participants_v1<-read_delim(paste0(config$alfa_dataset_raw, "Alfa Plus Long 2025__AlfaPlus_general_20250304095809(in).csv"), delim= ";", col_names = TRUE)%>%
  rename(subject = IdParticipante)%>%
  right_join(., alfa_subjects)%>%
  filter(sex==2)%>%
  dplyr::select(subject,Age_V1)%>%
  drop_na()%>%
  nrow()

alfa_participants_v2<-read_delim(paste0(config$alfa_dataset_raw, "Alfa Plus Long 2025__AlfaPlus_general_20250304095809(in).csv"), delim= ";", col_names = TRUE)%>%
  rename(subject = IdParticipante)%>%
  right_join(., alfa_subjects)%>%
  filter(sex==2)%>%
  dplyr::select(subject,Age_V2)%>%
  drop_na()%>%
  nrow()

alfa_subjects_dropout<-read_delim(paste0(config$alfa_dataset_raw, "Alfa Plus Long 2025__AlfaPlus_general_20250304095809(in).csv"), delim= ";", col_names = TRUE)%>%
rename(subject = IdParticipante)%>%
  right_join(., alfa_subjects)%>%
  filter(sex==2)%>%
  filter(is.na(Age_V2))%>%
  dplyr::select(subject)

dropout<-read_delim(paste0(config$alfa_dataset_raw, "Alfa Plus Long 2025_2010_STF_20250304095809(in).csv"), delim= ";", skip=1, col_names = TRUE, locale = locale(encoding = "Latin1") )%>%
  rename(subject = IdParticipante)%>%
  right_join(., alfa_subjects_dropout, join_by(subject))%>%
  dplyr::select(subject, STF_state, STF_reas, STF_reas5, STF_EspeIncumCri)%>%
  mutate(STF_reas = factor(STF_reas, levels = c("1", "2", "3", "4", "5", "6", "7"), labels = c("Death", "No response", "Participant withdrawal", "Clinician decision", "Promotor decision", "Protocol issue", "Other")),
         STF_state = factor(STF_state, levels = c("1", "2", "3", "4"), labels = c("Completed", "Failed preescreening", "Failed screening", "Discontinued")))

cognition1<-read_excel(paste0(config$alfa_dataset_cognition, "Visit 1 - Cognition Z Scores.xlsx"))%>%
  dplyr::select(-Gender, -Age_V1S1, -Years_Edu, -V1S1_dt, -Z_PACC_Composite_1.2._v1)%>%
  rename(memory = Z_Memory_Composite_v1, 
         attention = Z_Attention_Composite_v1,
         executive = Z_Executive_Composite_v1,
         language = Z_Language_Composite_v1,
         visual = Z_Visual_Composite_v1,
         subject = IdParticipante)%>%
  mutate(Visit=1)

cognition2<-read_excel(paste0(config$alfa_dataset_cognition, "Visit 2 - Cognition Z Scores.xlsx"))%>%
  dplyr::select(-Gender, -Age_V1S1, -Years_Edu, -V2S1_dt, -V1S1_dt, -Z_PACC_Composite_1.2._V2)%>%
  rename(memory = Z_Memory_Composite_V2, 
         attention = Z_Attention_Composite_V2,
         executive = Z_Executive_Composite_V2,
         language = Z_Language_Composite_V2,
         visual = Z_Visual_Composite_V2,
         subject = IdParticipante)%>%
   mutate(Visit=2)

cognition_wide<-bind_rows(cognition1, cognition2)%>%
  pivot_wider(values_from = c(attention, executive, memory, language, visual), names_from = Visit, names_prefix = "V")%>%
  dplyr::select(subject, ends_with("_V1"), ends_with("_V2"))%>%
  mutate(subject= as.numeric(subject))
  

ashs_wide<-ashs1%>%
  bind_rows(ashs1b)%>%
  bind_rows(ashs2)%>%
  mutate(subject = as.numeric(subject),
         vol_HIPP = vol_CA1+vol_CA2+vol_CA3+vol_DG+vol_SUB)%>%
  dplyr::select(-diff_mri)%>%
  filter(subject!= 21136,
         subject!= 12560,
         subject!= 12186)%>%
  pivot_wider(names_from = Visit, values_from = c(starts_with("vol_")))

biomarkers_wide<-biomarkers%>%
  dplyr::select(-A, -AT, -collection_dt, -SampleType)%>%
  pivot_wider(names_from = Visit, values_from = starts_with("CSF_"))

data_table<-data_naked%>%
  filter(sex==2)%>%
  left_join(., ashs_wide)%>%
  mutate(vol_HIPP_1 = ifelse(is.na(vol_HIPP_2), NA, vol_HIPP_1),
         vol_HIPP_2 = ifelse(is.na(vol_HIPP_1), NA, vol_HIPP_2),
         vol_tiv_1 =  ifelse(is.na(vol_HIPP_2), NA, vol_tiv_1))%>%
  left_join(.,cognition_wide)%>%
  mutate(attention_V1 = ifelse(is.na(attention_V2), NA, attention_V1),
         executive_V1 = ifelse(is.na(executive_V2), NA, executive_V1),
         memory_V1 = ifelse(is.na(memory_V2), NA, memory_V1),
         language_V1 = ifelse(is.na(language_V2), NA, language_V1),
         visual_V1 = ifelse(is.na(visual_V2), NA, visual_V1),)%>%
  left_join(.,biomarkers_wide)%>%
  dplyr::select(age_v1, kids_total, APOE_binary, education, vol_tiv_1, vol_HIPP_1, vol_HIPP_2, attention_V1, attention_V2, executive_V1, executive_V2, memory_V1, memory_V2, language_V1, language_V2, visual_V1, visual_V2, CSF_Abeta_ratio_V1, subject)%>%
  mutate(kids_total= ifelse(kids_total==4, "4+", as.character(kids_total)),
         vol_tiv_1 = vol_tiv_1/1000,
         vol_HIPP_1=vol_HIPP_1/1000,
         vol_HIPP_2=vol_HIPP_2/1000)%>%
  dplyr::select(age_v1, kids_total, education, APOE_binary, CSF_Abeta_ratio_V1, vol_HIPP_1, vol_HIPP_2, attention_V1, attention_V2, executive_V1, executive_V2, memory_V1, memory_V2, language_V1, language_V2, visual_V1, visual_V2, vol_tiv_1)

desc_table<-tbl_summary(data_table,
                        statistic = list(all_continuous() ~ "{mean} ({sd})"),
            by = kids_total, 
            missing="no",
            label= list(kids_total2 = "Number of childbirths",
                     APOE_binary = "APOE-E4 carriers, Count (%)",
                     age_v1 = "Age at first scan, Mean (SD)",
                     vol_tiv_1 = "Baseline Total Intracranial Volume (cm3), Mean (SD)",
                     education = "Education (years), Mean (SD)",
                     vol_HIPP_1 = "Baseline Hippocampal volume (cm3), Mean (SD)",
                     vol_HIPP_2 = "Follow-up Hippocampal volume (cm3), Mean (SD)",
                     attention_V1 = "Baseline composite attention score, Mean (SD)",
                     attention_V2 = "Follow-up composite attention score, Mean (SD)",
                     executive_V1 = "Baseline composite executive score, Mean (SD)",
                     executive_V2 = "Follow-up composite executive score, Mean (SD)",
                     memory_V1 = "Baseline composite memory score, Mean (SD)",
                     memory_V2 = "Follow-up composite memory score, Mean (SD)",
                     language_V1 = "Baseline composite language score, Mean (SD)",
                     language_V2 = "Follow-up composite language score, Mean (SD)",
                     visual_V1 = "Baseline composite visual score, Mean (SD)",
                     visual_V2 = "Follow-up composite visual score, Mean (SD)",
                     CSF_Abeta_ratio_V1 = "CSF Aβ42/40 ratio pg/ml, Mean (SD)"),
            value = list(APOE_binary= "1",
                         A= "1"))%>%
  add_n()%>%
  add_overall()%>%
  add_p()%>%
  modify_header(label = "**Variables**") %>% # update the column header
  bold_labels()


```

```{r, echo = F, warning=F}
desc_table
```

As shown in Table 1, 39 (15.3%) participants had no previous childbirths, 54 (21.2%) had one previous childbirth, 126 (49.6%) had 2, 27 (10.6%) had 3, and only 8 (3%) had 4 or more. Age, education and follow-up executive composite score significantly differed depending on parity. No differences were observed among groups for Aβ, APOE-ε4 carrier status, or HV. 

Carriers of APOE-E4 had significantly higher CSF Aβ42/40 load compared with non carriers (t = 5, df = 191, p < 0.001).

```{r, include=F}
t.test( CSF_Abeta_ratio_V1 ~ APOE_binary, data = data_table)
```


## Interplay between parity, APOE-E4 and HV

```{r, include = F}

data_ashs_female<-data_ashs_long%>%
  filter(sex==2)%>%
  group_by(subject)%>%
  filter(n()==2)%>%
  ungroup()%>%
  left_join(., A_long)%>%
  mutate(A = ifelse(CSF_Abeta_ratio<0.071, 1, 0))

mod_ashs_APOE<-lmer(vol_HIPP ~  kids_total*APOE_binary*diff_mri +age_v1 + scale(vol_tiv) + education  +(1|subject), data = data_ashs_female)
summary(mod_ashs_APOE)
plot(mod_ashs_APOE)
influenceIndexPlot(mod_ashs_APOE)

#-> Get all imputed datasets in long format
library(mice)
# Step 1: Set up and run imputation
pred <- make.predictorMatrix(data_ashs_female)
pred[, "subject"] <- -2
pred["subject", ] <- 0

method_vector <- make.method(data_ashs_female)
method_vector[names(data_ashs_female) != "CSF_Abeta_ratio"] <- ""
method_vector["CSF_Abeta_ratio"] <- "pmm"

imp_data <- mice(data_ashs_female, 
                 m = 5, 
                 method = method_vector, 
                 predictorMatrix = pred,
                 maxit = 50, 
                 seed = 500)

# Step 2: Fit model across all imputations (this is the correct way)
fit <- with(imp_data, 
            lmer(vol_HIPP ~ kids_total*APOE_binary*diff_mri + CSF_Abeta_ratio+
                 age_v1 + scale(vol_tiv) + education + 
                 (1|subject)))

# Step 3: Pool results using Rubin's rules
pooled <- pool(fit)
summary(pooled)

# If you want a tidier output
library(broom.mixed)
tidy(pooled, conf.int = TRUE)

###

mod_ashs_APOE_ab<-lmer(vol_HIPP ~  kids_total*APOE_binary*diff_mri +age_v1 + scale(vol_tiv) + CSF_Abeta_ratio+ education  +(1|subject), data = data_ashs_female)
summary(mod_ashs_APOE_ab)

mod_ashs_APOE_abneg<-lmer(vol_HIPP ~  kids_total*APOE_binary*diff_mri +age_v1 + scale(vol_tiv) + education  +(1|subject), data = data_ashs_female[data_ashs_female$A==0,])
summary(mod_ashs_APOE_abneg)

mod_ashs_APOE_A0<-lmer(vol_HIPP ~  kids_total*diff_mri +age_v1 + scale(vol_tiv) +education  +(1|subject), data = data_ashs_female[data_ashs_female$APOE_binary==0,])
summary(mod_ashs_APOE_A0)

mod_ashs_APOE_A1<-lmer(vol_HIPP ~  kids_total*diff_mri +age_v1 + scale(vol_tiv) +education  +(1|subject), data = data_ashs_female[data_ashs_female$APOE_binary==1,])
summary(mod_ashs_APOE_A1)

mod_ashs_APOE_A1_ab<-lmer(vol_HIPP ~  kids_total*diff_mri +age_v1 + scale(vol_tiv) +education + CSF_Abeta_ratio +(1|subject), data = data_ashs_female[data_ashs_female$APOE_binary==1,])
summary(mod_ashs_APOE_A1_ab)

mod_ashs_APOE_dicho<-lmer(vol_HIPP ~  kids_dicho*APOE_binary*diff_mri +age_v1 + scale(vol_tiv) + education  +(1|subject), data = data_ashs_female)
summary(mod_ashs_APOE_dicho)
ggpredict(mod_ashs_APOE_dicho, terms = c("APOE_binary", "diff_mri", "kids_dicho")) %>% plot()


```

Parity, APOE-ε4, and time showed a significant three-way interaction for total HV (β = `r tidy(mod_ashs_APOE)$estimate[11]`, SE = `r tidy(mod_ashs_APOE)$std.error[11]`, p = `r tidy(mod_ashs_APOE)$p.value[11]`), indicating that higher parity was associated with steeper decline in HV among APOE-ε4 carriers. The triple interaction effect was maintained when adjusting analyses by Aβ accumulation (β = `r tidy(mod_ashs_APOE_ab)$estimate[12]`, SE = `r tidy(mod_ashs_APOE_ab)$std.error[12]`, p = `r tidy(mod_ashs_APOE_ab)$p.value[12]`). Furthermore, the effect was maintained, although attenuated, when reproducing the analysis including only participants with negative Aβ status (CSF Aβ<0.071 pg/ml, n = 128) (β = `r tidy(mod_ashs_APOE_abneg)$estimate[11]`, SE = `r tidy(mod_ashs_APOE_abneg)$std.error[11]`, p = `r tidy(mod_ashs_APOE_abneg)$p.value[11]`). 

Stratifying by APOE-ε4 status yielded a significant interaction between parity and time in carriers only (β = `r tidy(mod_ashs_APOE_A1)$estimate[7]`, SE = `r tidy(mod_ashs_APOE_A1)$std.error[7]`, p = `r tidy(mod_ashs_APOE_A1)$p.value[7]`). Again, results were maintained when adjusting the model by Aβ accumulation (β = `r tidy(mod_ashs_APOE_A1_ab)$estimate[7]`, SE = `r tidy(mod_ashs_APOE_A1_ab)$std.error[7]`, p = `r tidy(mod_ashs_APOE_A1_ab)$p.value[7]`).

Reproducing the analysis using parity as a dichotomous variable reduced the 3-way interaction estimate and significance of the effect (β = `r tidy(mod_ashs_APOE_dicho)$estimate[11]`, SE = `r tidy(mod_ashs_APOE_dicho)$std.error[11]`, p = `r tidy(mod_ashs_APOE_dicho)$p.value[11]`).

```{r, include = F}
tiv_freesurfer<-read_excel(paste0(config$alfa_dataset_raw, "neuroimaging/cortical thickness/bx_freesurfer7_aparc_ALFA_PLUS_V1_20231120_20250703_110643.xlsx"))%>%
  filter(region=="eTIV")%>%
  dplyr::select(subject, value)%>%
  unique()%>%
  mutate(subject=as.numeric(subject),
         value=as.numeric(value))%>%
  rename(tiv = value)

gray_v1<-read_excel(paste0(config$alfa_dataset_raw, "neuroimaging/cortical thickness/bx_freesurfer7_aparc_ALFA_PLUS_V1_20231120_20250703_110643.xlsx"))%>%
  filter(measurement== "GrayVol")%>%
  dplyr::select(-ID, -measurement)%>%
  pivot_wider(names_from = region, values_from = value, names_prefix = "vol_")%>%
  mutate(across(starts_with("vol_"), ~as.numeric(.)))%>%
  group_by(subject)%>%
  summarise(across(starts_with("vol_"), sum))%>%
  mutate(Visit= "1",
         subject=as.numeric(subject),
         diff_mri= 0)

gray_v2<-read_excel(paste0(config$alfa_dataset_raw, "neuroimaging/cortical thickness/bx_freesurfer7_aparc_ALFA_PLUS_V2_20230518_20250703_111331.xlsx"))%>%
  filter(measurement== "GrayVol")%>%
  dplyr::select(-ID, -measurement)%>%
  pivot_wider(names_from = region, values_from = value, names_prefix = "vol_")%>%
  mutate(across(starts_with("vol_"), ~as.numeric(.)))%>%
  group_by(subject)%>%
  summarise(across(starts_with("vol_"), sum))%>%
  mutate(Visit= "2",
         subject=as.numeric(subject))%>%
  left_join(., diff_mri)

gray_all<-bind_rows(gray_v1, gray_v2)

data_gray<-left_join(data_naked, gray_all)%>%
  filter(sex==2)%>%
  left_join(., tiv_freesurfer)%>%
  group_by(subject)%>%
  filter(n()==2)%>%
  ungroup()%>%
  group_by(subject)%>%
  mutate(gray_all = rowSums(across(starts_with("vol_"))))

mod_cal<-lmer(vol_pericalcarine ~  kids_total*APOE_binary*diff_mri +age_v1 + scale(tiv) + education  +(1|subject), data = data_gray)
summary(mod_cal)

```

The 3-way interaction between APOE4 parity and time was not associated with GMV in the calcarine cortical area.

## Sensitivity analysis with hormonal variables
```{r, include=F}
data_ashs_female<-data_ashs_female%>%
  mutate(early_men = ifelse(age_menopause<45, 1, 0))

mod_ashs_APOE_horm<-lmer(vol_HIPP ~  kids_total2*APOE_binary*diff_mri +age_v1 + scale(vol_tiv) + education  + repro_span+ early_men+ HRT+ CSF_Abeta_ratio+ cause_menopause +(1|subject), data = data_ashs_female)
summary(mod_ashs_APOE_horm)
```

Adding other reproductive variables to our main analysis did not have an effect on the 3-way estimate.




## Cognitive trajectory 

```{r, include=F}

data_cog_composites<-bind_rows(cognition1, cognition2)%>%
  mutate(subject=as.numeric(subject),
         Visit=as.character(Visit))%>%
  right_join(., data_cog_long)%>%
  filter(sex==2)%>%
  group_by(subject)%>%
  filter(n()>1)%>%
  mutate(Visit=as.numeric(Visit))%>%
  left_join(., A_long)

#subject!=21136

fit_cog_att<-lmer(attention ~ time_diff*kids_total*APOE_binary +age_v1  +education +(1 | subject), data_cog_composites)
summary(fit_cog_att)
influenceIndexPlot(fit_cog_att)
plot(fit_cog_att)
plot_model(fit_cog_att, type = "int")

fit_cog_att_a0<-lmer(attention ~ time_diff*kids_total +age_v1  +education +(1 | subject), data_cog_composites[data_cog_composites$APOE_binary==0,])
plot_model(fit_cog_att_a0, type = "int")
#chose not to remove because appart from cook's distance everything seems fine.. fine in other domains too. But she is driving the results a lot so could report with and without?

ggplot(data_cog_composites, aes(x= time_diff, y= attention, group = subject, colour = APOE_binary))+
  geom_line()+
  facet_wrap(~kids_total)


fit_cog_ex<-lmer(executive ~ kids_total*time_diff*APOE_binary +age_v1  +education +(1 | subject), data_cog_composites)
summary(fit_cog_ex)
influenceIndexPlot(fit_cog_ex)

fit_cog_mem<-lmer(memory ~ kids_total*time_diff*APOE_binary +age_v1  +education +(1 | subject), data_cog_composites)
summary(fit_cog_mem)
influenceIndexPlot(fit_cog_mem)

fit_cog_lang<-lmer(language ~ kids_total*time_diff*APOE_binary +age_v1  +education +(1 | subject), data_cog_composites)
summary(fit_cog_lang)
influenceIndexPlot(fit_cog_lang)

fit_cog_vis<-lmer(visual ~ kids_total*time_diff*APOE_binary +age_v1  +education +(1 | subject), data_cog_composites)
summary(fit_cog_vis)
influenceIndexPlot(fit_cog_vis)


```
The interaction between parity, APOE4 and time yielded a weak trend which was not present after FDR correction. (not same direction so not sure if worth mentioning)

- talk about lower order interaction if present? they are a bit weird direction!


## Exploratory analysis of hippocampal subfields

```{r, include = F, warning=FALSE}

CA1_mean<-mean(data_ashs_female$vol_CA1[data_ashs_female$Visit==1], na.rm=T)
CA1_sd<-sd(data_ashs_female$vol_CA1[data_ashs_female$Visit==1], na.rm=T)
CA2_mean<-mean(data_ashs_female$vol_CA2[data_ashs_female$Visit==1], na.rm=T)
CA2_sd<-sd(data_ashs_female$vol_CA2[data_ashs_female$Visit==1], na.rm=T)
CA3_mean<-mean(data_ashs_female$vol_CA3[data_ashs_female$Visit==1], na.rm=T)
CA3_sd<-sd(data_ashs_female$vol_CA3[data_ashs_female$Visit==1], na.rm=T)
DG_mean<-mean(data_ashs_female$vol_DG[data_ashs_female$Visit==1], na.rm=T)
DG_sd<-sd(data_ashs_female$vol_DG[data_ashs_female$Visit==1], na.rm=T)
SUB_mean<-mean(data_ashs_female$vol_SUB[data_ashs_female$Visit==1], na.rm=T)
SUB_sd<-sd(data_ashs_female$vol_SUB[data_ashs_female$Visit==1], na.rm=T)

data_ashs_female<-data_ashs_female%>%
  mutate(vol_CA1_std = (vol_CA1 - CA1_mean) / CA1_sd,
         vol_CA2_std = (vol_CA2 - CA2_mean) / CA2_sd,
         vol_CA3_std = (vol_CA3 - CA3_mean) / CA3_sd,
         vol_DG_std = (vol_DG - DG_mean) / DG_sd,
         vol_SUB_std = (vol_SUB - SUB_mean) / SUB_sd)

mod_ashs_CA1<-lmer(vol_CA1_std ~  kids_total*diff_mri*APOE_binary +age_v1  + scale(vol_tiv) + education +(1|subject), data = data_ashs_female)
summary(mod_ashs_CA1)
influenceIndexPlot(mod_ashs_CA1)
mod_ashs_CA1_ab<-lmer(vol_CA1_std ~  kids_total*diff_mri*APOE_binary +age_v1  + CSF_Abeta_ratio+ scale(vol_tiv) + education +(1|subject), data = data_ashs_female)
summary(mod_ashs_CA1_ab)

mod_ashs_CA2<-lmer(vol_CA2_std ~ kids_total*diff_mri*APOE_binary + age_v1  + scale(vol_tiv) + education +(1|subject), data = data_ashs_female)
summary(mod_ashs_CA2)
influenceIndexPlot(mod_ashs_CA2)
mod_ashs_CA2_ab<-lmer(vol_CA2_std ~ kids_total*diff_mri*APOE_binary + age_v1  + scale(vol_tiv) + CSF_Abeta_ratio+  education +(1|subject), data = data_ashs_female)
summary(mod_ashs_CA2_ab)

mod_ashs_CA3<-lmer(vol_CA3_std ~  kids_total*diff_mri*APOE_binary +age_v1  + scale(vol_tiv) + education +(1|subject), data = data_ashs_female)
summary(mod_ashs_CA3)
influenceIndexPlot(mod_ashs_CA3)

mod_ashs_DG<-lmer(vol_DG_std ~  kids_total*diff_mri*APOE_binary +age_v1  + scale(vol_tiv) + education + (1|subject), data = data_ashs_female)
summary(mod_ashs_DG)
influenceIndexPlot(mod_ashs_DG)
mod_ashs_DG_ab<-lmer(vol_DG_std ~  kids_total*diff_mri*APOE_binary +age_v1  + scale(vol_tiv) + education + CSF_Abeta_ratio+ (1|subject), data = data_ashs_female)
summary(mod_ashs_DG_ab)

mod_ashs_SUB<-lmer(vol_SUB_std ~  kids_total*diff_mri*APOE_binary +age_v1  + scale(vol_tiv) + education +(1|subject), data = data_ashs_female)
summary(mod_ashs_SUB)
influenceIndexPlot(mod_ashs_SUB)


```


The 3-way interaction between parity, APOE4 and time was significant in the CA1 (β= `r tidy(mod_ashs_CA1)$estimate[11]`, SE = `r tidy(mod_ashs_CA1)$std.error[11]`, p= `r tidy(mod_ashs_CA1)$p.value[11]`), CA2  (β= `r tidy(mod_ashs_CA2)$estimate[11]`, SE = `r tidy(mod_ashs_CA2)$std.error[11]`, p= `r tidy(mod_ashs_CA2)$p.value[11]`) and in the DG (β= `r tidy(mod_ashs_DG)$estimate[11]`, SE = `r tidy(mod_ashs_DG)$std.error[11]`, p= `r tidy(mod_ashs_DG)$p.value[11]`). Similarly, parity was associated with steeper HV decline in APOE-E4 carriers.

Adjusting by AB....

## Exploratory analysis of inflammatory biomarkers

```{r, include=F}
data_biomarkers_long_female<-data_biomarkers_long%>%
  filter(sex==2)%>%
  group_by(subject)%>%
  filter(n()>1)
#not sure you are removing people who don't have more than 1 here

library(fitdistrplus)


gfap<-lmer(CSF_GFAP ~ kids_total*APOE_binary*biomarkers_time_diff + age_v1  + education +(1|subject), data = data_biomarkers_long_female)

summary(gfap) 

resid_gfap<-residuals(gfap)
fitted_gfap <- fitted(gfap)

bc_gfap <- MASS::boxcox(fitted_gfap + resid_gfap ~ 1)
lambda_gfap <- bc_gfap$x[which.max(bc_gfap$y)]

data_biomarkers_long_female$CSF_GFAP_transformed <- (data_biomarkers_long_female$CSF_GFAP^lambda_gfap - 1) / lambda_gfap

gfap2<-lmer(CSF_GFAP_transformed ~ kids_total*APOE_binary*biomarkers_time_diff + age_v1  + education +(1|subject), data = data_biomarkers_long_female)
summary(gfap2) 
plot(gfap2)
qqnorm(residuals(gfap2))

##########

strem<-lmer( CSF_sTREM2 ~ kids_total*APOE_binary*biomarkers_time_diff + age_v1  + education +(1|subject), data = data_biomarkers_long_female)

resid_strem<-residuals(strem)
fitted_strem <- fitted(strem)

bc_strem <- MASS::boxcox(fitted_strem + resid_strem ~ 1)
lambda_strem <- bc_strem$x[which.max(bc_strem$y)]

data_biomarkers_long_female$CSF_sTREM2_transformed <- (data_biomarkers_long_female$CSF_sTREM2^lambda_strem - 1) / lambda_strem

strem2<-lmer(CSF_sTREM2_transformed ~ kids_total*APOE_binary*biomarkers_time_diff + age_v1  + education +(1|subject), data = data_biomarkers_long_female)
summary(strem2) 
plot(strem2)
qqnorm(residuals(strem2))

######
il6<-lmer( `CSF_IL-6` ~ kids_total*APOE_binary*biomarkers_time_diff + age_v1  + education +(1|subject), data = data_biomarkers_long_female)

resid_il6<-residuals(il6)
fitted_il6 <- fitted(il6)

bc_il6 <- MASS::boxcox(fitted_il6 + resid_il6 ~ 1)
lambda_il6 <- bc_il6$x[which.max(bc_il6$y)]

data_biomarkers_long_female$`CSF_IL-6_transformed` <- (data_biomarkers_long_female$`CSF_IL-6`^lambda_il6 - 1) / lambda_il6

il62<-lmer(`CSF_IL-6_transformed` ~ kids_total*APOE_binary*biomarkers_time_diff + age_v1  + education +(1|subject), data = data_biomarkers_long_female)
summary(il62) 
plot(il62)
qqnorm(residuals(il62))

####

yk40<-lmer( CSF_YKL40 ~ kids_total*APOE_binary*biomarkers_time_diff + age_v1  + education +(1|subject), data = data_biomarkers_long_female)

resid_yk40<-residuals(yk40)
fitted_yk40 <- fitted(yk40)

bc_yk40 <- MASS::boxcox(fitted_yk40 + resid_yk40 ~ 1)
lambda_yk40 <- bc_yk40$x[which.max(bc_yk40$y)]

data_biomarkers_long_female$CSF_YKL40_transformed <- (data_biomarkers_long_female$CSF_YKL40^lambda_yk40 - 1) / lambda_yk40

yk402<-lmer(CSF_YKL40_transformed ~ kids_total*APOE_binary*biomarkers_time_diff + age_v1  + education +(1|subject), data = data_biomarkers_long_female)
summary(yk402) 
plot(yk402)
qqnorm(residuals(yk402))

#######

s100<-lmer( CSF_S100 ~ kids_total*APOE_binary*biomarkers_time_diff + age_v1  + education +(1|subject), data = data_biomarkers_long_female)

resid_s100<-residuals(s100)
fitted_s100 <- fitted(s100)

bc_s100 <- MASS::boxcox(fitted_s100 + resid_s100 ~ 1)
lambda_s100 <- bc_s100$x[which.max(bc_s100$y)]

data_biomarkers_long_female$CSF_S100_transformed <- (data_biomarkers_long_female$CSF_S100^lambda_s100 - 1) / lambda_s100

s1002<-lmer(CSF_S100_transformed ~ kids_total*APOE_binary*biomarkers_time_diff + age_v1  + education +(1|subject), data = data_biomarkers_long_female)
summary(s1002) 
plot(s1002)
qqnorm(residuals(s1002))

```

There was no association with any of the biomarkers assessed.


# Discussion

Our study aimed to investigate the impact of parity on hippocampal subfield volumes and relation to the APOE4 allele in post-menopausal women at risk of AD. Our results suggest that parity and AD specific genotype interact in women to impact hippocampal volume loss. In our sample, multiparous APOE4 non-carriers showed less hippocampus volume decline, whilst multiparous APOE4 carriers showed worse cognitive decline. These rseults show evidence of an interaction between parity and AD related genetic risk in women, which is associated in volume loss in a brain region which is a prime marker of AD. Essentially, this evidence strongly suggests there is a clear biological component to the interaction between parity and AD risk in women.

When using the categorical grouping for parity, results showed significantly bigger effect in multiparous women. This could indicate that the cumulative effects of parity are only significantly interact with genetic risk after having had more than one child. However, it is important to note that the multiparous groups had a fairly larger number of observations than the nulliparous or primiparous group, therefore lack of significance might indicate a power issue when performing the gorup interaction.

The AFLFA+ cohort is an ideal cohort to test this interaction, given the participants are enriched in genetic AD risk, As such, about 40% of our cohort is a carrier of the APOE4 allele, meaning group interactions are very balanced statistically.

Our findings reproduce the ones found by Lee and colleagues. Similarly, APOE4 carriers show detrimental effects of parity on hippocampal volume trajectory. Although we did not replicate their findings on memory, this could be due to the relative short amount of time between both visits, on 3 years in average. However, another hypothesis could be that there are intricate social factors at play in humans revolving around parity, which could be influencing cognitive resilience. A previous study done with the UK biobank also finds no effect of the interaction between APOE4 and parity on cognitive score (lindseth et al 2023), although their battery of test is limited.

Our post-hoc analysis of hippocampal subfields shows the most affected regions are the CA1, CA2, subiculum and enthorinal cortex, and a trend in the dentate gyrus. Given Bee's results, one could expect a stornger effect in the dentate gyrus, given most of the effect of parity in neurogenesis would reside in this subfield. The strong effect on parts of the cornu ammonis also reveals ageing mechanism.

Some limitations should be considered. Firstly, the number of follow-up visits is insufficient to observe nonlinear evolution, therefore limiting our understanding of possible mechanisms underlying AD pathogenesis. Future work should include additional time points to better understand the trajectory of these results. Secondly, although our sample’s excellent health record makes it ideal to identify subtle changes in the early AD continuum, the vast majority of ALFA+ study participants are white Caucasians, resulting in an ethnically homogeneous sample primarily from middle to upper-middle class backgrounds. Future work should reproduce these analyses in a more diverse cohort to understand the impact of ethnicity and social background on the present results. Lastly, the absence of direct hormonal measurements limits our ability to draw definitive conclusions about the role of sex steroid hormones in the observed associations. By relying on proxy measures, we can only estimate hormone-related effects, potentially leading to an underestimation of their actual influence.

# Figures

## Figure 1

```{r, echo=F, warning=F}
# Create grid of values for prediction - specify multiple time points

individual_slopes <- data_ashs_female %>%
  filter(!is.na(kids_total) & !is.na(APOE_binary) & !is.na(diff_mri) & !is.na(vol_HIPP)) %>%
  group_by(subject, kids_total, APOE_binary) %>%  # Only keep subjects with at least 2 observations
  summarise(
    individual_slope = coef(lm(vol_HIPP ~ diff_mri))[2],
    n_obs = n(),
    .groups = 'drop'
  ) %>%
  filter(!is.na(individual_slope))%>%
  mutate(APOE_binary = as.factor(APOE_binary))



time_range <- range(data_ashs_long$diff_mri, na.rm = TRUE)
time_seq <- seq(from = time_range[1], to = time_range[2])

emm_grid <- emmeans(mod_ashs_APOE, ~ diff_mri * kids_total * APOE_binary,
                    at = list(time = time_seq),
                    data= data_ashs_long)
emm_df <- as.data.frame(emm_grid)

slopes <- emtrends(mod_ashs_APOE, ~ kids_total * APOE_binary, var = "diff_mri", data= data_ashs_long, at = list(kids_total = c(0, 1, 2, 3, 4)))
slopes_df <- as.data.frame(slopes)

p1<-ggplot(slopes_df, aes(x = kids_total, y = diff_mri.trend, 
                            fill = as.factor(APOE_binary))) +
  geom_col(position = "dodge", alpha = 0.8, color = "black", size = 0.3) +
  geom_errorbar(aes(ymin = diff_mri.trend - SE, ymax = diff_mri.trend + SE),
                position = position_dodge(width = 0.9), width = 0.3)+
  th+
  labs(x = "Number of childbirths", 
     y = "Time slope", 
     color = expression(italic("APOE")*"-ε4"),
     fill = expression(italic("APOE")*"-ε4"))+
  geom_text(data = data.frame(parous_group = factor("Multiparous", 
                                                     levels = c("Nulliparous", "Primiparous", "Multiparous")), 
                              time_diff = Inf, 
                              Z_PACC = Inf, 
                              label = "p= 0.001**"),
            aes(x = time_diff, y = Z_PACC, label = label),
            size = 4, vjust = 1, hjust = 1, inherit.aes = FALSE)+
 geom_point(data = individual_slopes, 
             aes(x = kids_total, y = individual_slope, 
                 color = as.factor(APOE_binary)),
             position = position_jitterdodge(jitter.width = 0.2, dodge.width = 0.9),
             alpha = 0.4, size = 2, shape=21, colour="black") +
  scale_fill_manual(values = c("#1CA8B7", "#FF7043"),
                     labels = c("Non-Carriers", "Carriers"))+
scale_color_manual(values = c("#1CA8B7", "#FF7043"),
                     labels = c("Non-Carriers", "Carriers"))+
  scale_x_continuous(
    breaks = c(0, 1, 2, 3, 4),
    labels = c("0", "1", "2", "3", "4+")
  )
```

```{r, echo=F, warning=F}

#install.packages("ggtext")
library(ggtext)

# Get slopes across parity values for each APOE group
parity_range <- c(0, 1, 2, 3, 4)

# Non-carriers
slopes_nc <- emtrends(mod_ashs_APOE_A0,
                      specs = ~ kids_total,
                      var = "diff_mri",
                      at = list(kids_total = parity_range),
                      data=data_ashs_female[data_ashs_female$APOE_binary==0,])
slopes_nc_df <- as.data.frame(slopes_nc)
slopes_nc_df$APOE_binary <- "Non-carriers"

# Carriers  
slopes_c <- emtrends(mod_ashs_APOE_A1,
                     specs = ~ kids_total,
                     var = "diff_mri",
                     at = list(kids_total = parity_range),
                     data=data_ashs_female[data_ashs_female$APOE_binary==1,])
slopes_c_df <- as.data.frame(slopes_c)
slopes_c_df$APOE_binary <- "Carriers"

# Combine
all_slopes <- rbind(slopes_nc_df, slopes_c_df)

# Same data as above, but on one plot
p2<-all_slopes%>%
  mutate(APOE_binary=factor(APOE_binary, levels = c("Non-carriers", "Carriers")))%>%
ggplot( aes(x = kids_total, y = diff_mri.trend)) +
  geom_ribbon(aes(ymin = lower.CL, ymax = upper.CL, fill = APOE_binary), 
              alpha = 0.3) +
  geom_line(aes(color = APOE_binary), size = 1) +
  geom_hline(yintercept = 0, linetype = "dashed", color = "gray50") +
  facet_wrap(~APOE_binary) +
  labs(x = "Parity (number of births)",
       y = "Estimated slope of HV change (cm3)") +
  th +
  theme(legend.position = "none")+
  geom_text(data = data.frame(APOE_binary = factor("Non-carriers", 
                                                     levels = c("Non-carriers", "Carriers")), 
                              kids_total = Inf, 
                              diff_mri.trend = Inf, 
                              label = "p=0.257"),
            aes(x = kids_total, y = diff_mri.trend, label = label),
            size = 4, vjust = 3, hjust = 1.2, inherit.aes = FALSE)+
  geom_text(data = data.frame(APOE_binary = factor("Carriers", 
                                                     levels = c("Non-carriers", "Carriers")), 
                              kids_total = Inf, 
                              diff_mri.trend = Inf, 
                              label = "p=0.001**"),
            aes(x = kids_total, y = diff_mri.trend, label = label),
            size = 4, vjust = 3, hjust = 1.2, inherit.aes = FALSE)+
  scale_color_manual(values = c("#1CA8B7", "#FF7043"),
                     labels = c("Non-Carriers", "Carriers"))+
  scale_fill_manual(values = c("#1CA8B7", "#FF7043"),
                     labels = c("Non-Carriers", "Carriers"))+
  scale_x_continuous(
    breaks = c(0, 1, 2, 3, 4),
    labels = c("0", "1", "2", "3", "4+")
  )

```

```{r, echo=F, warning=F}
p1/p2+
  plot_annotation(tag_levels = 'A')
```

## Figure 2

```{r, echo=F, warning=F}
# Standardize outcome variables based on visit 1 values
# First, calculate mean and SD from visit 1 only



# Fit models with standardized outcomes - APOE = 0
models_apoe0 <- list(
  CA1 = lmer(vol_CA1_std ~ kids_total2*diff_mri + vol_tiv + age_v1 + education + (1 | subject),
             data = filter(data_ashs_female, APOE_binary == 0)),
  CA2 = lmer(vol_CA2_std ~ kids_total2*diff_mri + vol_tiv + age_v1 + education + (1 | subject),
             data = filter(data_ashs_female, APOE_binary == 0)),
  CA3 = lmer(vol_CA3_std ~ kids_total2*diff_mri + vol_tiv + age_v1 + education + (1 | subject),
             data = filter(data_ashs_female, APOE_binary == 0)),
  DG = lmer(vol_DG_std ~ kids_total2*diff_mri + vol_tiv + age_v1 + education + (1 | subject),
            data = filter(data_ashs_female, APOE_binary == 0)),
  SUB = lmer(vol_SUB_std ~ kids_total2*diff_mri + vol_tiv + age_v1 + education + (1 | subject),
             data = filter(data_ashs_female, APOE_binary == 0))
)

# Fit models with standardized outcomes - APOE = 1
models_apoe1 <- list(
  CA1 = lmer(vol_CA1_std ~ kids_total2*diff_mri + vol_tiv + age_v1 + education + (1 | subject),
             data = filter(data_ashs_female, APOE_binary == 1)),
  CA2 = lmer(vol_CA2_std ~ kids_total2*diff_mri + vol_tiv + age_v1 + education + (1 | subject),
             data = filter(data_ashs_female, APOE_binary == 1)),
  CA3 = lmer(vol_CA3_std ~ kids_total2*diff_mri + vol_tiv + age_v1 + education + (1 | subject),
             data = filter(data_ashs_female, APOE_binary == 1)),
  DG = lmer(vol_DG_std ~ kids_total2*diff_mri + vol_tiv + age_v1 + education + (1 | subject),
            data = filter(data_ashs_female, APOE_binary == 1)),
  SUB = lmer(vol_SUB_std ~ kids_total2*diff_mri + vol_tiv + age_v1 + education + (1 | subject),
             data = filter(data_ashs_female, APOE_binary == 1))
)

# Extract coefficients for the interaction term
extract_interaction <- function(model_list, apoe_status) {
  map_df(names(model_list), function(region) {
    model <- model_list[[region]]
    coef_df <- tidy(model, conf.int = TRUE) %>%
      filter(term == "kids_total2:diff_mri") %>%
      mutate(region = region,
             APOE = apoe_status)
    return(coef_df)
  })
}

results_apoe0 <- extract_interaction(models_apoe0, "APOE = 0")
results_apoe1 <- extract_interaction(models_apoe1, "APOE = 1")

# Combine results
forest_data <- bind_rows(results_apoe0, results_apoe1) %>%
  mutate(
    region = factor(region, levels = rev(c("CA1", "CA2", "CA3", "DG", "SUB"))),
    # Add significance stars
    sig_label = case_when(
      p.value < 0.001 ~ "***",
      p.value < 0.01 ~ "**",
      p.value < 0.05 ~ "*",
      TRUE ~ ""
    )
  )

# Create forest plot with standardized estimates
forest1<-ggplot(forest_data, aes(x = estimate, y = region, color = APOE)) +
  geom_vline(xintercept = 0, linetype = "dashed", color = "gray50") +
  geom_errorbarh(aes(xmin = conf.low, xmax = conf.high), 
                 height = 0.2, position = position_dodge(width = 0.5)) +
  geom_point(size = 3, position = position_dodge(width = 0.5)) +
  geom_text(aes(label = sig_label), 
            position = position_dodge(width = 0.5),
            vjust = -0.8, size = 4, show.legend = FALSE) +
  labs(x = "Parity:Time",
       y = "Hippocampal subfields",
       color = "APOE-e4") +
  th+
  theme(legend.position = "none")
```

```{r, echo=F, warning=F}

# Fit models with standardized outcomes - APOE = 0
models_apoe0 <- list(
  CA1 = lmer(vol_CA1_std ~ kids_total2*diff_mri + vol_tiv + age_v1 + education + CSF_Abeta_ratio+ (1 | subject),
             data = filter(data_ashs_female, APOE_binary == 0)),
  CA2 = lmer(vol_CA2_std ~ kids_total2*diff_mri + vol_tiv + age_v1 + education + CSF_Abeta_ratio+ (1 | subject),
             data = filter(data_ashs_female, APOE_binary == 0)),
  CA3 = lmer(vol_CA3_std ~ kids_total2*diff_mri + vol_tiv + age_v1 + education + CSF_Abeta_ratio+ (1 | subject),
             data = filter(data_ashs_female, APOE_binary == 0)),
  DG = lmer(vol_DG_std ~ kids_total2*diff_mri + vol_tiv + age_v1 + education + CSF_Abeta_ratio+ (1 | subject),
            data = filter(data_ashs_female, APOE_binary == 0)),
  SUB = lmer(vol_SUB_std ~ kids_total2*diff_mri + vol_tiv + age_v1 + education + CSF_Abeta_ratio+ (1 | subject),
             data = filter(data_ashs_female, APOE_binary == 0))
)

# Fit models with standardized outcomes - APOE = 1
models_apoe1 <- list(
  CA1 = lmer(vol_CA1_std ~ kids_total2*diff_mri + vol_tiv + age_v1 + education + CSF_Abeta_ratio+ (1 | subject),
             data = filter(data_ashs_female, APOE_binary == 1)),
  CA2 = lmer(vol_CA2_std ~ kids_total2*diff_mri + vol_tiv + age_v1 + education + CSF_Abeta_ratio+ (1 | subject),
             data = filter(data_ashs_female, APOE_binary == 1)),
  CA3 = lmer(vol_CA3_std ~ kids_total2*diff_mri + vol_tiv + age_v1 + education + CSF_Abeta_ratio+ (1 | subject),
             data = filter(data_ashs_female, APOE_binary == 1)),
  DG = lmer(vol_DG_std ~ kids_total2*diff_mri + vol_tiv + age_v1 + education + CSF_Abeta_ratio+ (1 | subject),
            data = filter(data_ashs_female, APOE_binary == 1)),
  SUB = lmer(vol_SUB_std ~ kids_total2*diff_mri + vol_tiv + age_v1 + education + CSF_Abeta_ratio+ (1 | subject),
             data = filter(data_ashs_female, APOE_binary == 1))
)

# Extract coefficients for the interaction term
extract_interaction <- function(model_list, apoe_status) {
  map_df(names(model_list), function(region) {
    model <- model_list[[region]]
    coef_df <- tidy(model, conf.int = TRUE) %>%
      filter(term == "kids_total2:diff_mri") %>%
      mutate(region = region,
             APOE = apoe_status)
    return(coef_df)
  })
}

results_apoe0 <- extract_interaction(models_apoe0, "APOE = 0")
results_apoe1 <- extract_interaction(models_apoe1, "APOE = 1")

# Combine results
forest_data <- bind_rows(results_apoe0, results_apoe1) %>%
  mutate(
    region = factor(region, levels = rev(c("CA1", "CA2", "CA3", "DG", "SUB"))),
    # Add significance stars
    sig_label = case_when(
      p.value < 0.001 ~ "***",
      p.value < 0.01 ~ "**",
      p.value < 0.05 ~ "*",
      TRUE ~ ""
    )
  )

# Create forest plot with standardized estimates
forest2<-ggplot(forest_data, aes(x = estimate, y = region, color = APOE)) +
  geom_vline(xintercept = 0, linetype = "dashed", color = "gray50") +
  geom_errorbarh(aes(xmin = conf.low, xmax = conf.high), 
                 height = 0.2, position = position_dodge(width = 0.5)) +
  geom_point(size = 3, position = position_dodge(width = 0.5)) +
  geom_text(aes(label = sig_label), 
            position = position_dodge(width = 0.5),
            vjust = -0.8, size = 4, show.legend = FALSE) +
  labs(x = "Parity:Interaction + Aβ load",
       y = "Hippocampal subfields",
       color = "APOE-e4") +
  th+
  theme(
    axis.title.y = element_blank(),
    axis.text.y = element_blank(),
    axis.ticks.y = element_blank(),
    axis.line.y = element_blank() # Removes the axis line itself (depending on the theme used)
  )
```

```{r, echo=F, warning=FALSE}
forest1+forest2
```

# Appendix

## MRI data acquisition

MRI scans were acquired on a in-house 3.0 T scanner (GE Discovery MR750 W 3 T) using the same protocol for all participants, which included one T1- and three T2- weighted sequences. The 3D-T1w sequence was designed with an isotropic voxel size of1mm3 and a matrix size of256x256x160 (TR/TE/TI = 8.0/3.7/450 ms, NSA = 1, flip angle = 8°). Three 3D-T2w sequences, with a voxel size of1 mm× 1 mm× 3 mm, were also used: fluid attenuation inversion recovery (FLAIR: TR/TE/ TI = 11,000/90/2600 ms, flip angle = 160°), fast spin echo (FSE: TR/TE = 5000/85 ms, flip angle = 110°) and, gradient echo (GRE: TR/TE = 1300/23 ms, flip angle = 15°). All scans were visually assessed to verify their quality and to detect incidental findings by a trained neuroradiologist and have been reported elsewhere (Brugulat-Serrat et al. 2017). At visit 1, ten participants were excluded due to the presence of a meningioma, as well as 37 participants due to susceptibility, motion artefacts or segmentation problems, resulting in a total of 561 images available for subsequent analysis. At Visit 2, ... The medial temporal lobe atrophy was assessed by Medial Temporal Atrophy scale [@scheltens1992a].

# Miscellaneous

```{r, jacks, include=F, eval=F}
jack1<-read_excel(paste0(config$alfa_dataset_neuroimaging, "Jack signature/bx_signature_jack_ALFA_PLUS_V1_20231120_20250527_142751.xlsx"))%>%
  filter(weighted == "TRUE")%>%
  dplyr::select(-ID, -weighted, -signature)%>%
  pivot_wider(values_from = value, names_from = measurement)%>%
  mutate(Visit = 1)

jack2<-read_excel(paste0(config$alfa_dataset_neuroimaging, "Jack signature/bx_signature_jack_ALFA_PLUS_V2_20230518_20250210_114442.xlsx"))%>%
  filter(weighted == "TRUE")%>%
  dplyr::select(-ID, -weighted, -signature)%>%
  pivot_wider(values_from = value, names_from = measurement)%>%
  mutate(Visit = 2)

data_jack<-bind_rows(jack1, jack2)%>%
  mutate(subject=as.numeric(subject))%>%
  right_join(.,data_naked)%>%
  left_join(., diff_mri)

mod_jack<-lmer(GrayVol ~  kids_total*diff_mri*APOE_binary +age_v1 + education  +(1|subject), data = data_jack[data_jack$sex==2, ])
summary(mod_jack)
```

```{r graph subfields, echo=F, warning=F, include=F, eval=F}
preds_CA1 <- ggpredict(mod_ashs_CA1, terms = c("diff_mri", "kids_total", "APOE_binary"))
preds_CA2 <- ggpredict(mod_ashs_CA2, terms = c("diff_mri", "kids_total", "APOE_binary"))
preds_SUB <- ggpredict(mod_ashs_SUB, terms = c("diff_mri", "kids_total", "APOE_binary"))
preds_ERC <- ggpredict(mod_ashs_ERC, terms = c("diff_mri", "kids_total", "APOE_binary"))
preds_DG <- ggpredict(mod_ashs_DG, terms = c("diff_mri", "kids_total", "APOE_binary"))

preds_CA1_A0<-as.data.frame(preds_CA1)%>%
  rename(kids_total_factor = group,
         APOE_binary=facet)%>%
  filter(APOE_binary==0)%>%
  mutate(APOE_binary = "Non-Carriers",,
         kids_total_factor = ifelse(kids_total_factor == 4, "4+", as.character(kids_total_factor)))


preds_CA1_A1<-as.data.frame(preds_CA1)%>%
  rename(kids_total_factor = group,
         APOE_binary=facet)%>%
  filter(APOE_binary==1)%>%
  mutate(APOE_binary = "Carriers",
         kids_total_factor = ifelse(kids_total_factor == 4, "4+", as.character(kids_total_factor)))

preds_CA2_A0<-as.data.frame(preds_CA2)%>%
  rename(kids_total_factor = group,
         APOE_binary=facet)%>%
  filter(APOE_binary==0)%>%
  mutate(APOE_binary = "Non-Carriers",,
         kids_total_factor = ifelse(kids_total_factor == 4, "4+", as.character(kids_total_factor)))


preds_CA2_A1<-as.data.frame(preds_CA2)%>%
  rename(kids_total_factor = group,
         APOE_binary=facet)%>%
  filter(APOE_binary==1)%>%
  mutate(APOE_binary = "Carriers",
         kids_total_factor = ifelse(kids_total_factor == 4, "4+", as.character(kids_total_factor)))

preds_SUB_A0<-as.data.frame(preds_SUB)%>%
  rename(kids_total_factor = group,
         APOE_binary=facet)%>%
  filter(APOE_binary==0)%>%
  mutate(APOE_binary = "Non-Carriers",,
         kids_total_factor = ifelse(kids_total_factor == 4, "4+", as.character(kids_total_factor)))


preds_SUB_A1<-as.data.frame(preds_SUB)%>%
  rename(kids_total_factor = group,
         APOE_binary=facet)%>%
  filter(APOE_binary==1)%>%
  mutate(APOE_binary = "Carriers",
         kids_total_factor = ifelse(kids_total_factor == 4, "4+", as.character(kids_total_factor)))

preds_ERC_A0<-as.data.frame(preds_ERC)%>%
  rename(kids_total_factor = group,
         APOE_binary=facet)%>%
  filter(APOE_binary==0)%>%
  mutate(APOE_binary = "Non-Carriers",,
         kids_total_factor = ifelse(kids_total_factor == 4, "4+", as.character(kids_total_factor)))


preds_ERC_A1<-as.data.frame(preds_ERC)%>%
  rename(kids_total_factor = group,
         APOE_binary=facet)%>%
  filter(APOE_binary==1)%>%
  mutate(APOE_binary = "Carriers",
         kids_total_factor = ifelse(kids_total_factor == 4, "4+", as.character(kids_total_factor)))

preds_DG_A0<-as.data.frame(preds_DG)%>%
  rename(kids_total_factor = group,
         APOE_binary=facet)%>%
  filter(APOE_binary==0)%>%
  mutate(APOE_binary = "Non-Carriers",,
         kids_total_factor = ifelse(kids_total_factor == 4, "4+", as.character(kids_total_factor)))


preds_DG_A1<-as.data.frame(preds_DG)%>%
  rename(kids_total_factor = group,
         APOE_binary=facet)%>%
  filter(APOE_binary==1)%>%
  mutate(APOE_binary = "Carriers",
         kids_total_factor = ifelse(kids_total_factor == 4, "4+", as.character(kids_total_factor)))


CA1_plot<-data_ashs %>%
  filter(sex == 2, !is.na(APOE_binary)) %>%
  mutate(kids_total_factor = ifelse(kids_total == 4, "4+", as.character(kids_total)),
         APOE_binary = factor(APOE_binary, levels = c("0", "1"), labels = c("Non-Carriers", "Carriers"))) %>%
  ggplot(aes(x = diff_mri, y = vol_CA1, color = kids_total_factor)) +
  geom_line(data =preds_CA1_A0, aes(x=x, 
                                y=predicted, 
                                group=kids_total_factor), size=1.3) +
  geom_line(data =preds_CA1_A1, aes(x=x, 
                                y=predicted, 
                                group=kids_total_factor), size=1.3) +
  geom_ribbon(data =preds_CA1_A0, aes(x=x,  
                                  ymin= conf.low, 
                                  ymax = conf.high, 
                                  group=kids_total_factor), inherit.aes = F, alpha=.07)+
  geom_ribbon(data =preds_CA1_A1, aes(x=x,  
                                  ymin= conf.low, 
                                  ymax = conf.high, 
                                  group=kids_total_factor), inherit.aes = F, alpha=.07)+
  facet_wrap(~APOE_binary) +
  scale_color_manual(values = c("0" = "#8AD9B5", "1" = "#50B88E", "2" = "#047A56", "3" = "#0A5940", "4+" = "#0F3F2D")) +
  th +
  labs(x = "Time difference (years)", 
       y = "CA1", 
       color = "Number of childbirths")+
  theme(legend.position = "none")

CA2_plot<-data_ashs %>%
  filter(sex == 2, !is.na(APOE_binary)) %>%
  mutate(kids_total_factor = ifelse(kids_total == 4, "4+", as.character(kids_total)),
         APOE_binary = factor(APOE_binary, levels = c("0", "1"), labels = c("Non-Carriers", "Carriers"))) %>%
  ggplot(aes(x = diff_mri, y = vol_CA2, color = kids_total_factor)) +
  geom_line(data =preds_CA2_A0, aes(x=x, 
                                y=predicted, 
                                group=kids_total_factor), size=1.3) +
  geom_line(data =preds_CA2_A1, aes(x=x, 
                                y=predicted, 
                                group=kids_total_factor), size=1.3) +
  geom_ribbon(data =preds_CA2_A0, aes(x=x,  
                                  ymin= conf.low, 
                                  ymax = conf.high, 
                                  group=kids_total_factor), inherit.aes = F, alpha=.07)+
  geom_ribbon(data =preds_CA2_A1, aes(x=x,  
                                  ymin= conf.low, 
                                  ymax = conf.high, 
                                  group=kids_total_factor), inherit.aes = F, alpha=.07)+
  facet_wrap(~APOE_binary) +
  scale_color_manual(values = c("0" = "#8AD9B5", "1" = "#50B88E", "2" = "#047A56", "3" = "#0A5940", "4+" = "#0F3F2D")) +
  th +
  labs(x = "Time difference (years)", 
       y = "CA2", 
       color = "Number of childbirths")+
  theme(legend.position = "none")

SUB_plot<-data_ashs %>%
  filter(sex == 2, !is.na(APOE_binary)) %>%
  mutate(kids_total_factor = ifelse(kids_total == 4, "4+", as.character(kids_total)),
         APOE_binary = factor(APOE_binary, levels = c("0", "1"), labels = c("Non-Carriers", "Carriers"))) %>%
  ggplot(aes(x = diff_mri, y = vol_SUB, color = kids_total_factor)) +
  geom_line(data =preds_SUB_A0, aes(x=x, 
                                y=predicted, 
                                group=kids_total_factor), size=1.3) +
  geom_line(data =preds_SUB_A1, aes(x=x, 
                                y=predicted, 
                                group=kids_total_factor), size=1.3) +
  geom_ribbon(data =preds_SUB_A0, aes(x=x,  
                                  ymin= conf.low, 
                                  ymax = conf.high, 
                                  group=kids_total_factor), inherit.aes = F, alpha=.07)+
  geom_ribbon(data =preds_SUB_A1, aes(x=x,  
                                  ymin= conf.low, 
                                  ymax = conf.high, 
                                  group=kids_total_factor), inherit.aes = F, alpha=.07)+
  facet_wrap(~APOE_binary) +
  scale_color_manual(values = c("0" = "#8AD9B5", "1" = "#50B88E", "2" = "#047A56", "3" = "#0A5940", "4+" = "#0F3F2D")) +
  th +
  labs(x = "Time difference (years)", 
       y = "SUB", 
       color = "Number of childbirths")+
  theme(legend.position = "none")

ERC_plot<-data_ashs %>%
  filter(sex == 2, !is.na(APOE_binary)) %>%
  mutate(kids_total_factor = ifelse(kids_total == 4, "4+", as.character(kids_total)),
         APOE_binary = factor(APOE_binary, levels = c("0", "1"), labels = c("Non-Carriers", "Carriers"))) %>%
  ggplot(aes(x = diff_mri, y = vol_ERC, color = kids_total_factor)) +
  geom_line(data =preds_ERC_A0, aes(x=x, 
                                y=predicted, 
                                group=kids_total_factor), size=1.3) +
  geom_line(data =preds_ERC_A1, aes(x=x, 
                                y=predicted, 
                                group=kids_total_factor), size=1.3) +
  geom_ribbon(data =preds_ERC_A0, aes(x=x,  
                                  ymin= conf.low, 
                                  ymax = conf.high, 
                                  group=kids_total_factor), inherit.aes = F, alpha=.07)+
  geom_ribbon(data =preds_ERC_A1, aes(x=x,  
                                  ymin= conf.low, 
                                  ymax = conf.high, 
                                  group=kids_total_factor), inherit.aes = F, alpha=.07)+
  facet_wrap(~APOE_binary) +
  scale_color_manual(values = c("0" = "#8AD9B5", "1" = "#50B88E", "2" = "#047A56", "3" = "#0A5940", "4+" = "#0F3F2D")) +
  th +
  labs(x = "Time difference (years)", 
       y = "ERC", 
       color = "Number of childbirths")+
  theme(legend.position = "none")

DG_plot<-data_ashs %>%
  filter(sex == 2, !is.na(APOE_binary)) %>%
  mutate(kids_total_factor = ifelse(kids_total == 4, "4+", as.character(kids_total)),
         APOE_binary = factor(APOE_binary, levels = c("0", "1"), labels = c("Non-Carriers", "Carriers"))) %>%
  ggplot(aes(x = diff_mri, y = vol_DG, color = kids_total_factor)) +
  geom_line(data =preds_DG_A0, aes(x=x, 
                                y=predicted, 
                                group=kids_total_factor), size=1.3) +
  geom_line(data =preds_DG_A1, aes(x=x, 
                                y=predicted, 
                                group=kids_total_factor), size=1.3) +
  geom_ribbon(data =preds_DG_A0, aes(x=x,  
                                  ymin= conf.low, 
                                  ymax = conf.high, 
                                  group=kids_total_factor), inherit.aes = F, alpha=.07)+
  geom_ribbon(data =preds_DG_A1, aes(x=x,  
                                  ymin= conf.low, 
                                  ymax = conf.high, 
                                  group=kids_total_factor), inherit.aes = F, alpha=.07)+
  facet_wrap(~APOE_binary) +
  scale_color_manual(values = c("0" = "#8AD9B5", "1" = "#50B88E", "2" = "#047A56", "3" = "#0A5940", "4+" = "#0F3F2D")) +
  th +
  labs(x = "Time difference (years)", 
       y = "DG", 
       color = "Number of childbirths")
  


(CA1_plot + CA2_plot ) / 
(ERC_plot + SUB_plot) 
```

```{r graph whole hippocampus, eval=F}

preds_ashs <- ggpredict(mod_ashs_APOE, terms = c("diff_mri", "kids_total", "APOE_binary"))

preds_ashs_A0<-as.data.frame(preds_ashs)%>%
  rename(kids_total_factor = group,
         APOE_binary=facet)%>%
  filter(APOE_binary==0)%>%
  mutate(APOE_binary = "Non-Carrier",
         kids_total_factor= ifelse(as.character(kids_total_factor)==4, "4+", as.character(kids_total_factor)))

preds_ashs_A1<-as.data.frame(preds_ashs)%>%
  rename(kids_total_factor = group,
         APOE_binary=facet)%>%
  filter(APOE_binary==1)%>%
  mutate(APOE_binary = "Carrier",
         kids_total_factor= ifelse(as.character(kids_total_factor)==4, "4+", as.character(kids_total_factor)))

# Create a combined factor for color mapping
data_plot <- data_ashs %>%
  filter(sex == 2, !is.na(APOE_binary)) %>%
  mutate(kids_total_factor = ifelse(kids_total==4, "4+", kids_total),
         APOE_binary = factor(APOE_binary, levels = c("0", "1"), labels = c("Non-Carrier", "Carrier")),
         color_group = paste(APOE_binary, kids_total_factor, sep = "_"))

preds_ashs_A0 <- preds_ashs_A0 %>%
  mutate(color_group = paste("Non-Carrier", kids_total_factor, sep = "_"))

preds_ashs_A1 <- preds_ashs_A1 %>%
  mutate(color_group = paste("Carrier", kids_total_factor, sep = "_"))


# Define color palettes
color_values <- c(
  "Non-Carrier_0" = "#4DD0E1", "Non-Carrier_1" = "#1CA8B7", "Non-Carrier_2" = "#1E8895", "Non-Carrier_3" = "#0D707B", "Non-Carrier_4+" = "#0F565E",
  "Carrier_0" = "#FF8A65", "Carrier_1" = "#FF7043", "Carrier_2" = "#FF5722", "Carrier_3" = "#F4511E", "Carrier_4+" = "#BF360C"
)

ggplot(data_plot, aes(x = diff_mri, y = vol_HIPP, color = color_group)) +
  geom_point(alpha = 0.3, size=2) +
  geom_line(data = preds_ashs_A0, aes(x=x, y=predicted, group=kids_total_factor, color=color_group), size=1.3) +
  geom_line(data = preds_ashs_A1, aes(x=x, y=predicted, group=kids_total_factor, color=color_group), size=1.3) +
  geom_ribbon(data = preds_ashs_A0, aes(x=x, ymin=conf.low, ymax=conf.high, 
                                         group=kids_total_factor, fill=color_group), 
              inherit.aes = F, alpha=.15)+
  geom_ribbon(data = preds_ashs_A1, aes(x=x, ymin=conf.low, ymax=conf.high, 
                                         group=kids_total_factor, fill=color_group), 
              inherit.aes = F, alpha=.15)+
  facet_wrap(~APOE_binary) +
  scale_color_manual(values = color_values, labels = c("0", "1", "2", "3", "4+", "0", "1", "2", "3", "4+")) +
  scale_fill_manual(values = color_values, labels = c("0", "1", "2", "3", "4+", "0", "1", "2", "3", "4+")) +
  labs(x = "Time difference (days)", 
       y = "PACC Z-score", 
       color = "Number of childbirths",
       fill = "Number of childbirths")+
  th
```

```{r power analysis, include=F, eval=F}

#power analysis if needed
sd_kids_total <- sd(data_ashs$kids_total[data_ashs$sex==2], na.rm=TRUE)
prop_A <- prop.table(table(data_ashs$APOE_binary[data_ashs$sex==2]))
p_A <- 0.488 
sd_A <- sqrt(p_A * (1 - p_A))

(34.677 * (sd_kids_total * sd_A) / sd(data_ashs$vol_HIPP[data_ashs$sex==2], na.rm=TRUE))^2

power.t.regression(beta = 34.677,
                   sd.predictor = sd_kids_total * sd_A,
                   sd.outcome = sd(data_ashs$vol_HIPP[data_ashs$sex==2], na.rm=TRUE),
                   r.squared = (34.677 * (sd_kids_total * sd_A) / sd(data_ashs$vol_HIPP[data_ashs$sex==2], na.rm=TRUE))^2,
                   null.beta = 0, 
                   k.total = 8, 
                   n=196, 
                   alpha=0.05)

```

```{r test , eval = F, include = F}
vif(mod_ashs_APOE)

mod_ashs_APOE<-lmer(vol_HIPP ~  kids_total*APOE_binary*diff_mri +age_v1 + scale(vol_tiv) + education  +(1|subject), data = data_ashs[data_ashs$sex==2 & data_ashs$A==0,])
summary(mod_ashs_APOE)


plot_model(mod_ashs_APOE, type = "int")


data_ashs_wide <- data_ashs %>%
  dplyr::select(subject, sex, age_v1, kids_total, APOE_binary, education, starts_with("vol_"), Visit, A, CSF_Abeta_ratio) %>%
  drop_na(Visit) %>%
  pivot_wider(names_from = Visit, values_from = starts_with("vol_")) %>%
  left_join(., date_mri_long)



data_ashs_wide <- data_ashs %>%
  dplyr::select(subject, sex, age_v1, kids_total, APOE_binary, education, starts_with("vol_"), Visit, A, CSF_Abeta_ratio) %>%
  drop_na(Visit) %>%
  pivot_wider(names_from = Visit, values_from = starts_with("vol_")) %>%
  left_join(., date_mri_long) %>%
  {
    df <- .
    vol_1_cols <- grep("^vol_.*_1$", names(df), value = TRUE)
    
    for (col_1 in vol_1_cols) {
      base_name <- sub("_1$", "", col_1)
      col_2 <- paste0(base_name, "_2")
      
      if (col_2 %in% names(df)) {
        df[[paste0(base_name, "_diff")]] <- (df[[col_2]] - df[[col_1]]) / df$diff_mri
      }
    }
    df
  }


test<-lm(vol_HIPP_diff ~ kids_total*APOE_binary + age_v1 + education + scale(vol_tiv_2), data = data_ashs_wide[data_ashs_wide$sex==2 & data_ashs_wide$A==0,])

summary(test)

data_ashs%>%
  filter(APOE_binary==1, A==1, sex==2)%>%
  dplyr::select(subject)%>%
  unique%>%
  nrow()

data_ashs <- data_ashs %>%
  mutate(apoeAB = case_when(
    APOE_binary == 0 & A == 0 ~ 1,
    APOE_binary == 0 & A == 1 ~ 2,
    APOE_binary == 1 & A == 0 ~ 3,
    APOE_binary == 1 & A == 1 ~ 4
  ))%>%
  mutate(apoeAB = factor(apoeAB,levels = c(1,2,3,4)))

test1<-lmer(vol_HIPP ~ kids_total*apoeAB*diff_mri + age_v1 + education + scale(vol_tiv) + (1|subject), data = data_ashs[data_ashs$sex==2,])

# Get predicted values
pred <- ggpredict(test1, terms = c("kids_total [all]", "apoeAB [1,2,3,4]", "diff_mri"))

# Plot with custom colors
ggplot(pred, aes(x = x, y = predicted, color = factor(group), linetype = factor(facet))) +
  geom_line(linewidth = 1) +
  geom_ribbon(aes(ymin = conf.low, ymax = conf.high, fill = factor(group)), 
              alpha = 0.1, color = NA) +
  scale_color_manual(values = c("#1b9e77", "#d95f02", "#7570b3", "#e7298a"),
                     labels = c("APOE4-/AB-", "APOE4-/AB+", "APOE4+/AB-", "APOE4+/AB+")) +
  scale_fill_manual(values = c("#1b9e77", "#d95f02", "#7570b3", "#e7298a"),
                    labels = c("APOE4-/AB-", "APOE4-/AB+", "APOE4+/AB-", "APOE4+/AB+")) +
  facet_wrap(~facet, labeller = labeller(facet = c("Low parity", "High parity"))) +
  labs(x = "Time", y = "Hippocampal Volume", 
       color = "Group", fill = "Group") +
  theme_minimal()
```
