<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.6.40">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Clazz">

<title>Parity and AD specific neurodegeneration</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="paper_ashs_pregnancy_APOE_files/libs/clipboard/clipboard.min.js"></script>
<script src="paper_ashs_pregnancy_APOE_files/libs/quarto-html/quarto.js"></script>
<script src="paper_ashs_pregnancy_APOE_files/libs/quarto-html/popper.min.js"></script>
<script src="paper_ashs_pregnancy_APOE_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="paper_ashs_pregnancy_APOE_files/libs/quarto-html/anchor.min.js"></script>
<link href="paper_ashs_pregnancy_APOE_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="paper_ashs_pregnancy_APOE_files/libs/quarto-html/quarto-syntax-highlighting-549806ee2085284f45b00abea8c6df48.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="paper_ashs_pregnancy_APOE_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="paper_ashs_pregnancy_APOE_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="paper_ashs_pregnancy_APOE_files/libs/bootstrap/bootstrap-8a79a254b8e706d3c925cde0a310d4f0.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">


</head>

<body>

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
  <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#abstract" id="toc-abstract" class="nav-link active" data-scroll-target="#abstract"><span class="header-section-number">1</span> Abstract</a></li>
  <li><a href="#introduction" id="toc-introduction" class="nav-link" data-scroll-target="#introduction"><span class="header-section-number">2</span> Introduction</a></li>
  <li><a href="#methods" id="toc-methods" class="nav-link" data-scroll-target="#methods"><span class="header-section-number">3</span> Methods</a>
  <ul class="collapse">
  <li><a href="#study-population" id="toc-study-population" class="nav-link" data-scroll-target="#study-population"><span class="header-section-number">3.1</span> Study population</a></li>
  <li><a href="#reproductive-variables" id="toc-reproductive-variables" class="nav-link" data-scroll-target="#reproductive-variables"><span class="header-section-number">3.2</span> Reproductive variables</a></li>
  <li><a href="#cognitive-measures" id="toc-cognitive-measures" class="nav-link" data-scroll-target="#cognitive-measures"><span class="header-section-number">3.3</span> Cognitive measures</a></li>
  <li><a href="#ad-biomarkers" id="toc-ad-biomarkers" class="nav-link" data-scroll-target="#ad-biomarkers"><span class="header-section-number">3.4</span> AD biomarkers</a></li>
  <li><a href="#imaging-data-acquisition-and-preprocessing" id="toc-imaging-data-acquisition-and-preprocessing" class="nav-link" data-scroll-target="#imaging-data-acquisition-and-preprocessing"><span class="header-section-number">3.5</span> Imaging data acquisition and preprocessing</a></li>
  <li><a href="#statistical-analyses" id="toc-statistical-analyses" class="nav-link" data-scroll-target="#statistical-analyses"><span class="header-section-number">3.6</span> Statistical analyses</a></li>
  </ul></li>
  <li><a href="#results" id="toc-results" class="nav-link" data-scroll-target="#results"><span class="header-section-number">4</span> Results</a>
  <ul class="collapse">
  <li><a href="#descriptive-results" id="toc-descriptive-results" class="nav-link" data-scroll-target="#descriptive-results"><span class="header-section-number">4.1</span> Descriptive results</a></li>
  <li><a href="#interplay-between-parity-apoe-e4-and-hv" id="toc-interplay-between-parity-apoe-e4-and-hv" class="nav-link" data-scroll-target="#interplay-between-parity-apoe-e4-and-hv"><span class="header-section-number">4.2</span> Interplay between parity, APOE-E4 and HV</a></li>
  <li><a href="#sensitivity-analysis-with-hormonal-variables" id="toc-sensitivity-analysis-with-hormonal-variables" class="nav-link" data-scroll-target="#sensitivity-analysis-with-hormonal-variables"><span class="header-section-number">4.3</span> Sensitivity analysis with hormonal variables</a></li>
  <li><a href="#cognitive-trajectory" id="toc-cognitive-trajectory" class="nav-link" data-scroll-target="#cognitive-trajectory"><span class="header-section-number">4.4</span> Cognitive trajectory</a></li>
  <li><a href="#exploratory-analysis-of-hippocampal-subfields" id="toc-exploratory-analysis-of-hippocampal-subfields" class="nav-link" data-scroll-target="#exploratory-analysis-of-hippocampal-subfields"><span class="header-section-number">4.5</span> Exploratory analysis of hippocampal subfields</a></li>
  <li><a href="#exploratory-analysis-of-inflammatory-biomarkers" id="toc-exploratory-analysis-of-inflammatory-biomarkers" class="nav-link" data-scroll-target="#exploratory-analysis-of-inflammatory-biomarkers"><span class="header-section-number">4.6</span> Exploratory analysis of inflammatory biomarkers</a></li>
  </ul></li>
  <li><a href="#discussion" id="toc-discussion" class="nav-link" data-scroll-target="#discussion"><span class="header-section-number">5</span> Discussion</a></li>
  <li><a href="#figures" id="toc-figures" class="nav-link" data-scroll-target="#figures"><span class="header-section-number">6</span> Figures</a>
  <ul class="collapse">
  <li><a href="#figure-1" id="toc-figure-1" class="nav-link" data-scroll-target="#figure-1"><span class="header-section-number">6.1</span> Figure 1</a></li>
  <li><a href="#figure-2" id="toc-figure-2" class="nav-link" data-scroll-target="#figure-2"><span class="header-section-number">6.2</span> Figure 2</a></li>
  </ul></li>
  <li><a href="#appendix" id="toc-appendix" class="nav-link" data-scroll-target="#appendix"><span class="header-section-number">7</span> Appendix</a>
  <ul class="collapse">
  <li><a href="#mri-data-acquisition" id="toc-mri-data-acquisition" class="nav-link" data-scroll-target="#mri-data-acquisition"><span class="header-section-number">7.1</span> MRI data acquisition</a></li>
  </ul></li>
  <li><a href="#miscellaneous" id="toc-miscellaneous" class="nav-link" data-scroll-target="#miscellaneous"><span class="header-section-number">8</span> Miscellaneous</a></li>
  </ul>
</nav>
</div>
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Parity and AD specific neurodegeneration</h1>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>Clazz </p>
          </div>
  </div>
    
  
    
  </div>
  


</header>


<section id="abstract" class="level1" data-number="1">
<h1 data-number="1"><span class="header-section-number">1</span> Abstract</h1>
<p>300 words</p>
</section>
<section id="introduction" class="level1" data-number="2">
<h1 data-number="2"><span class="header-section-number">2</span> Introduction</h1>
<p>Alzheimer’s disease (AD) is a neurodegenerative disorder characterised by the accumulation of pathophysiological abnormalities leading to neurodegeneration and progressive cognitive and functional decline. Ageing, carring the apolipoprotein E E4 (APOE4) and having female chromosomal sex are three established non-modifiable risk factors for the typical late-onset of this disease <span class="citation" data-cites="riedel2016">[<a href="#ref-riedel2016" role="doc-biblioref">1</a>]</span>. Carrying one copy of APOE4 increases AD risk by three to four fold, whilst carrying 2 copies further increases risk by 10 folds <span class="citation" data-cites="corder1993">[<a href="#ref-corder1993" role="doc-biblioref">2</a>]</span>. Women have a higher incidence of late-onset AD dementia <span class="citation" data-cites="beam2018">[<a href="#ref-beam2018" role="doc-biblioref">3</a>]</span> and show faster cognitive decline after diagnosis of MCI or AD dementia <span class="citation" data-cites="sexdiff2018">[<a href="#ref-sexdiff2018" role="doc-biblioref">4</a>]</span>. Amongst APOE4 carriers, women exhibit an increased risk of conversion from healthy ageing to MCI, or from MCI to AD dementia between the ages of 65 to 75 <span class="citation" data-cites="Altmann2014 Neu2017">[<a href="#ref-Altmann2014" role="doc-biblioref">5</a>,<a href="#ref-Neu2017" role="doc-biblioref">6</a>]</span>. Therefore, studying the interactive effects of these non-modifiable risk factors might help us better understand the reason behind the aforementioned sex differences in AD incidence and trajectory.</p>
<p>The hippocampus is one of the first regions affected by AD, and whole hippocampal volume is a hallmark predictor of conversion from MCI to AD <span class="citation" data-cites="apostolova2006">[<a href="#ref-apostolova2006" role="doc-biblioref">7</a>]</span>. APOE4 carriers show reduced HV compared with non carriers [citation]. Older EPOE4 carriers show reduced hippocampal volume compared with non-carriers, especially in the CA1 and subiculum tissue <span class="citation" data-cites="pievani2011">[<a href="#ref-pievani2011" role="doc-biblioref">8</a>]</span>.</p>
<ul>
<li>mention info on DG here?</li>
</ul>
<p>Interestingly, this structure is deeply impacted by pregnancy in a lasting way <span class="citation" data-cites="puri2023">[<a href="#ref-puri2023" role="doc-biblioref">9</a>]</span>, but long-term effects remain vastly misunderstood. Animal models show decreased hippocampal neurogenesis and HV during gestation and postpartum period <span class="citation" data-cites="pawluski2007 eid2019">[<a href="#ref-pawluski2007" role="doc-biblioref">10</a>,<a href="#ref-eid2019" role="doc-biblioref">11</a>]</span>, but parous middle aged rats show increased hipocampal neurogenesis compared with nulliparous <span class="citation" data-cites="lee2024">[<a href="#ref-lee2024" role="doc-biblioref">12</a>]</span>. In humans, pregnancy is associated with overall gray matter volume loss, and hippocampal volume loss <span class="citation" data-cites="Hoekzema2017">[<a href="#ref-Hoekzema2017" role="doc-biblioref">13</a>]</span>. These changes are detectable up to 6 years post-partum in primiparous women <span class="citation" data-cites="martínez-garcía2021">[<a href="#ref-martínez-garcía2021" role="doc-biblioref">14</a>]</span>. However, a study using 19,787 middle-aged CU women from the UK biobank found that middle-aged parous women exhibited reduced brain age compared with nulliparous women based on gray matter volume patterns <span class="citation" data-cites="de2020">[<a href="#ref-de2020" role="doc-biblioref">15</a>]</span>. This effect was mainly driven by anatomical components situated in the limbic region, like the hippocampus and parahippocampal regions, putamen, amygdala and accumbens. This study also found significant positive linear associations between the Putamen, Hippocampus, Amygdala and Accumbens and number of childbirths.</p>
<ul>
<li>repport effect sizes on the de Lange study to compare with yours?</li>
</ul>
<p>Evidence on how previous pregnancies influence cognitive decline and AD risk is conflicting. Amongst epidemiological studies that explored a potential association between parity and AD, some report a link between multiparity and greater cognitive decline <span class="citation" data-cites="colucci2006">[<a href="#ref-colucci2006" role="doc-biblioref">16</a>]</span>, whilst other report no link or the opposite findings <span class="citation" data-cites="Fox2018 Fox2013 wolfova2024">[<a href="#ref-Fox2018" role="doc-biblioref">17</a>,<a href="#ref-Fox2013" role="doc-biblioref">18</a>.,<a href="#ref-wolfova2024" role="doc-biblioref">19</a>]</span>Adittionally, a post-mortem histopathological study of 42 women, with clinical diagnosis spanning the spectrum from normal aging to definite AD, found that multiparity was associated with a greater number of neuritic plaques <span class="citation" data-cites="Beeri2009a">[<a href="#ref-Beeri2009a" role="doc-biblioref">20</a>]</span>. So although parity seems to be assoicated with better neuroimaging outcomes, links with cognitive oucomes are mixed.</p>
<p>Lee and colleagues <span class="citation" data-cites="lee2024">[<a href="#ref-lee2024" role="doc-biblioref">12</a>]</span> propose that this “parity paradox” could be reconciled using the Healthy cell bias hypothesis, where hormonal changes brought on by prgenancy could impact cell metabolism differently depending on presence of disease risk. Using wildtype and APOE4 models middle aged female rats, they show that APOE status modifies the effect of parity on neurogenesis and HV. Parity was associated with neurogenesis in wildtype rats, particularly in the ventral dentate gyrus, but decreased neurogenesis in APOE4 rats. In addition to this, parous APOE4 rats showed a spatial solving strategy which indicated a decreased recruitment of the hippocampus. To our knowledge, no study has explored this interaction of APOE4 and parity on hippocampal structure in humans.</p>
<p>Our objective was to explore potential links and mechanisms through which pregnancy interacts with AD genetc risk in CU postmenopausal women at risk of AD. We investigated the interplay between parity, APOE-E4 carriership, HV and… to test whether AD related genetic risk modifies the effect of parity on HV trajectory in middle aged women. We hypothesized that parous women would show decreased whole hippocampal volume loss but this relationship would be inversed in APOE4 carriers.</p>
</section>
<section id="methods" class="level1" data-number="3">
<h1 data-number="3"><span class="header-section-number">3</span> Methods</h1>
<section id="study-population" class="level2" data-number="3.1">
<h2 data-number="3.1" class="anchored" data-anchor-id="study-population"><span class="header-section-number">3.1</span> Study population</h2>
<p>We included postmenopausal women who completed the baseline and first longitudinal visit (with an average time between visits of 3.32 years) from the ongoing ALFA+ study (Clinicaltrials.gov, NCT01835717). ALFA+ is a research cohort of middle-aged cognitively unimpaired participants, many of whom are offspring of AD patients, who have been deeply characterized by clinical interviews, lifestyle and risk factor questionnaires, cognitive testing, CSF biomarkers, and neuroimaging procedures, including magnetic resonance imaging (MRI), and Aβ positron emission tomography (PET). All of these procedures are repeated every 3 years with the main aim of identifying the earliest pathophysiological changes in the preclinical AD continuum <span class="citation" data-cites="molinuevo2016">[<a href="#ref-molinuevo2016" role="doc-biblioref">21</a>]</span>. Briefly, ALFA+ inclusion criteria were: 1) participants who had previously participated in the 45-65/FPM2012 study (ALFA parent cohort); 2) age between 45 and 75 years at the moment of inclusion in the cohort; 3) long-term commitment to undergo all tests and study procedures (MRI, PET, and lumbar puncture). Exclusion criteria at baseline included: 1) cognitive impairment (Clinical Dementia Rating [CDR] &gt; 0, Mini-Mental State Examination [MMSE] &lt; 26, semantic fluency &lt; 12); 2) any significant systemic illness or unstable medical condition which could lead to difficulty complying with the protocol; 3) any contraindication to any test or procedure; 4) family history of monogenic AD. In addition, women with a history of cardiovascular-related gestational disease, such as preeclampsia, were excluded from our sample for analysis. Sex and menopausal status, defined as absence of menstruations for 12 consecutive months, were self disclosed. Women who reported being premenopausal or perimenopausal were not included in our analysis either.</p>
<p>The study was approved by the Independent Ethics Committee “Parc de Salut Mar”, Barcelona, and all participants gave written informed consent. Baseline visits took place between 2016 and 2019, and the follow-up visit between 2019 and 2022.</p>
<p>The sample consisted of 254 participants at baseline visit, and 211 at follow-up (Table 1). Out of the 43 participants who dropped out of the study between visits, 30 withdrew for personal reasons, 3 became unresponsive, 2 died, 5 showed incidental l findings on their structural MRI, and 3 were excluded due to mild cognitive impairment clinical diagnosis.</p>
</section>
<section id="reproductive-variables" class="level2" data-number="3.2">
<h2 data-number="3.2" class="anchored" data-anchor-id="reproductive-variables"><span class="header-section-number">3.2</span> Reproductive variables</h2>
<p>Parity was defined as the number of biological children and used as a continuous and dichotomous [parous vs nulliparous] variable in the main analyses. Due to the small number of grand multiparous participants, anyone with 4 or more biological children was grouped together. Hormonal Replacement Therapy (HRT) use was defined as a binary variable [currently using vs not currently using]. Cause of menopause was also defined as a binary variable [natural vs non-natural]. Reproductive span was calculated by subtracting age at menarche from age at menopause. Early menopause was defined as a binary variable [age at menopause&gt; 45 vs age at menopause≤ 45]. These informations were self-reported by participants during an interview with a clinician at the baseline visit. When asking about age at menopause, participants were asked to report their age 6(?) months after their last period.</p>
</section>
<section id="cognitive-measures" class="level2" data-number="3.3">
<h2 data-number="3.3" class="anchored" data-anchor-id="cognitive-measures"><span class="header-section-number">3.3</span> Cognitive measures</h2>
<p>Cognition was defined using 5 composite scores that measured sparate congitive domains categorised as follows: attention, executive functions, memory, language and visual processing. Z-scores were computed for Visit 1 and 2 and used in the linear regressions.</p>
</section>
<section id="ad-biomarkers" class="level2" data-number="3.4">
<h2 data-number="3.4" class="anchored" data-anchor-id="ad-biomarkers"><span class="header-section-number">3.4</span> AD biomarkers</h2>
<p>CSF collection, processing, and storage in the ALFA+ study have been described previously <span class="citation" data-cites="Milà-Alomà2020">[<a href="#ref-Milà-Alomà2020" role="doc-biblioref">22</a>]</span>. Briefly, CSF Aβ42 and Aβ40 were measured with the exploratory Roche NeuroToolKit immunoassays (Roche Diagnostics International Ltd, Rotkreuz, Switzerland) on a cobas e 601 module. Measurements were performed at the Clinical Neurochemistry Laboratory, Sahlgrenska University Hospital, Molndal, Sweden. Aß accumulation (Aß+, Aß–) was estimated using CSF Aß42/40. Aß status (Aß+, Aß–) was defined using the cutoff of 0.071 for the ratio Aß42/40 [23], where any participant below this cutoff was considered to be Aβ+.</p>
<ul>
<li>add other inflammation biomarkers</li>
</ul>
</section>
<section id="imaging-data-acquisition-and-preprocessing" class="level2" data-number="3.5">
<h2 data-number="3.5" class="anchored" data-anchor-id="imaging-data-acquisition-and-preprocessing"><span class="header-section-number">3.5</span> Imaging data acquisition and preprocessing</h2>
<p>Automatic Segmentation of Hippocampal Subfields (ASHS) software <span class="citation" data-cites="Yushkevich2015">[<a href="#ref-Yushkevich2015" role="doc-biblioref">23</a>]</span> was used on the T1, T2 and inversion recovery (IR) weighted images to segment the hippocampal formation in the following sub-regions: Brodmann areas 35 and 36, cornu Ammonis (CA) 1, 2 and 3, dentate gyrus (DG), entorhinal cortex (ERC), parahippocampal cortex (PHC), subiculum (SUB), and the hippocampal sulcus (SUL). All segmentations were visually inspected before proceeding with the statistical analyses.</p>
<p>Hippocampal values are reported in cm3 and were obtained at both baseline and follow-up visits.</p>
<p>Gray matter volume was obtained using Freesurfer7.</p>
<p>Total Intracranial volume (TIV) was estimated using deformation fields obtained during the nonlinear registration of each subject’s T1-weighted MRI to a whole-brain template. The FSL Brain Extraction Tool <span class="citation" data-cites="smith2002">[<a href="#ref-smith2002" role="doc-biblioref">24</a>]</span> was applied to the template to create an intracranial mask, which was warped into each subject’s MRI at native space to obtain the TIV measurement.</p>
</section>
<section id="statistical-analyses" class="level2" data-number="3.6">
<h2 data-number="3.6" class="anchored" data-anchor-id="statistical-analyses"><span class="header-section-number">3.6</span> Statistical analyses</h2>
<p>We used linear mixed-effects models to assess the joint effects of parity and APOE-e4 (exposures), on HV and cognition z-score (outcomes) over both visits, and test if this association is dependent on time. These models included a three-way interaction term among parity, APOE4, and time (the interval between Visit 1 and Visit 2 in years) and all lower-order interactions. Cognitive analyses were adjusted for years of education, and age at baseline visit. HV analyses were additionally adjusted for TIV.</p>
<p>Analyses that showed a significant parity x APOE-e4 interaction were reproduced adding Aβ load as a covariate to disentangle the neuropathological contributions of Aβ accumulation from the phenogenetic contributions of APOE4. Given the high correlation between Aβ load and APOE4 variables, collinearity was tested for each model and can be found in the suppplementary materials. In addition to this, we reproduced the analysis in Aβ-negative participants only.</p>
<p>Specificity of the results was tested by testing the 3-way interaction between parity, APOE4 and time on gray matter volume changes in the calcarine cortex.</p>
<p>Sensitivity analyses were performed to explore the impact of additional reproductive variables on HV in the context of the 3-way interaction between parity APOE4 and time. These regressions were further adjusted by HRT, early menopause, reproductive span and cause of menopause.</p>
<p>Additional exploratory linear regressions were also performed to explore the interaction between parity and APOE-e4 positivity on hippocampal subfield volumes across visits. Subfields values are standardized. Given the exploratory nature of these analyses, results are reported without correction for multiple comparisons.</p>
<p>Multiple imputation was used on covariates with missing data in order to avoid imcomplete case deletion.</p>
<p>Statistical significance was set at p &lt; 0.05. Analyses were conducted using R software, version 4.5.1 (The R Foundation for Statistical Computing, Vienna, Austria). The package ‘emmeans’ was used to calculate estimated marginal mean for direct group comparison of predicted effects <span class="citation" data-cites="lenth2017">[<a href="#ref-lenth2017" role="doc-biblioref">25</a>]</span>. The package tidiverse was used for data processing and plotting <span class="citation" data-cites="wickham2019">[<a href="#ref-wickham2019" role="doc-biblioref">26</a>]</span>. The package gtsummary was used for formatting tables <span class="citation" data-cites="sjoberg2021">[<a href="#ref-sjoberg2021" role="doc-biblioref">27</a>]</span>. The package ggeffects was used to extract predicted values from regression models <span class="citation" data-cites="lüdecke2018">[<a href="#ref-lüdecke2018" role="doc-biblioref">28</a>]</span>. The package lme4 was used for performing mixed effect linear regression models <span class="citation" data-cites="bates2015">[<a href="#ref-bates2015" role="doc-biblioref">29</a>]</span>. The package MICE was used for multiple imputation of missing data.</p>
<p>Rates of missing data in exposures or outcome variables at each time point are reported (Table 1).</p>
<ul>
<li>cognitive subfield analyses are given standardized</li>
<li>analysis with biomarker as outcome were normalised using box-cox regression</li>
</ul>
</section>
</section>
<section id="results" class="level1" data-number="4">
<h1 data-number="4"><span class="header-section-number">4</span> Results</h1>
<section id="descriptive-results" class="level2" data-number="4.1">
<h2 data-number="4.1" class="anchored" data-anchor-id="descriptive-results"><span class="header-section-number">4.1</span> Descriptive results</h2>
<div class="cell">
<div class="cell-output-display">
<div id="hybxwehhne" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#hybxwehhne table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#hybxwehhne thead, #hybxwehhne tbody, #hybxwehhne tfoot, #hybxwehhne tr, #hybxwehhne td, #hybxwehhne th {
  border-style: none;
}

#hybxwehhne p {
  margin: 0;
  padding: 0;
}

#hybxwehhne .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 16px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#hybxwehhne .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#hybxwehhne .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#hybxwehhne .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#hybxwehhne .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#hybxwehhne .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#hybxwehhne .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#hybxwehhne .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#hybxwehhne .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#hybxwehhne .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#hybxwehhne .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#hybxwehhne .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#hybxwehhne .gt_spanner_row {
  border-bottom-style: hidden;
}

#hybxwehhne .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#hybxwehhne .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#hybxwehhne .gt_from_md > :first-child {
  margin-top: 0;
}

#hybxwehhne .gt_from_md > :last-child {
  margin-bottom: 0;
}

#hybxwehhne .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#hybxwehhne .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#hybxwehhne .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#hybxwehhne .gt_row_group_first td {
  border-top-width: 2px;
}

#hybxwehhne .gt_row_group_first th {
  border-top-width: 2px;
}

#hybxwehhne .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#hybxwehhne .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#hybxwehhne .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#hybxwehhne .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#hybxwehhne .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#hybxwehhne .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#hybxwehhne .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#hybxwehhne .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#hybxwehhne .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#hybxwehhne .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#hybxwehhne .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#hybxwehhne .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#hybxwehhne .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#hybxwehhne .gt_left {
  text-align: left;
}

#hybxwehhne .gt_center {
  text-align: center;
}

#hybxwehhne .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#hybxwehhne .gt_font_normal {
  font-weight: normal;
}

#hybxwehhne .gt_font_bold {
  font-weight: bold;
}

#hybxwehhne .gt_font_italic {
  font-style: italic;
}

#hybxwehhne .gt_super {
  font-size: 65%;
}

#hybxwehhne .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#hybxwehhne .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#hybxwehhne .gt_indent_1 {
  text-indent: 5px;
}

#hybxwehhne .gt_indent_2 {
  text-indent: 10px;
}

#hybxwehhne .gt_indent_3 {
  text-indent: 15px;
}

#hybxwehhne .gt_indent_4 {
  text-indent: 20px;
}

#hybxwehhne .gt_indent_5 {
  text-indent: 25px;
}

#hybxwehhne .katex-display {
  display: inline-flex !important;
  margin-bottom: 0.75em !important;
}

#hybxwehhne div.Reactable > div.rt-table > div.rt-thead > div.rt-tr.rt-tr-group-header > div.rt-th-group:after {
  height: 0px !important;
}
</style>

<table class="gt_table caption-top table table-sm table-striped small" data-quarto-postprocess="true" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
<colgroup>
<col style="width: 11%">
<col style="width: 11%">
<col style="width: 11%">
<col style="width: 11%">
<col style="width: 11%">
<col style="width: 11%">
<col style="width: 11%">
<col style="width: 11%">
<col style="width: 11%">
</colgroup>
<thead>
<tr class="gt_col_headings header">
<th id="label" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col"><strong>Variables</strong></th>
<th id="n" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><strong>N</strong></th>
<th id="stat_0" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><strong>Overall</strong><br>
N = 254<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height:0;"><sup>1</sup></span></th>
<th id="stat_1" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><strong>0</strong><br>
N = 39<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height:0;"><sup>1</sup></span></th>
<th id="stat_2" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><strong>1</strong><br>
N = 54<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height:0;"><sup>1</sup></span></th>
<th id="stat_3" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><strong>2</strong><br>
N = 126<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height:0;"><sup>1</sup></span></th>
<th id="stat_4" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><strong>3</strong><br>
N = 27<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height:0;"><sup>1</sup></span></th>
<th id="stat_5" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><strong>4+</strong><br>
N = 8<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height:0;"><sup>1</sup></span></th>
<th id="p.value" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><strong>p-value</strong><span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height:0;"><sup>2</sup></span></th>
</tr>
</thead>
<tbody class="gt_table_body">
<tr class="odd">
<td class="gt_row gt_left" headers="label" style="font-weight: bold">Age at first scan, Mean (SD)</td>
<td class="gt_row gt_center" headers="n">254</td>
<td class="gt_row gt_center" headers="stat_0">60.8 (4.9)</td>
<td class="gt_row gt_center" headers="stat_1">58.8 (4.5)</td>
<td class="gt_row gt_center" headers="stat_2">59.7 (4.9)</td>
<td class="gt_row gt_center" headers="stat_3">61.5 (4.8)</td>
<td class="gt_row gt_center" headers="stat_4">61.7 (4.8)</td>
<td class="gt_row gt_center" headers="stat_5">65.2 (4.2)</td>
<td class="gt_row gt_center" headers="p.value">&lt;0.001</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label" style="font-weight: bold">Education (years), Mean (SD)</td>
<td class="gt_row gt_center" headers="n">254</td>
<td class="gt_row gt_center" headers="stat_0">13.0 (3.6)</td>
<td class="gt_row gt_center" headers="stat_1">14.6 (3.7)</td>
<td class="gt_row gt_center" headers="stat_2">12.7 (3.2)</td>
<td class="gt_row gt_center" headers="stat_3">12.6 (3.6)</td>
<td class="gt_row gt_center" headers="stat_4">13.6 (3.8)</td>
<td class="gt_row gt_center" headers="stat_5">10.9 (3.1)</td>
<td class="gt_row gt_center" headers="p.value">0.013</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label" style="font-weight: bold">APOE-E4 carriers, Count (%)</td>
<td class="gt_row gt_center" headers="n">254</td>
<td class="gt_row gt_center" headers="stat_0">130 (51%)</td>
<td class="gt_row gt_center" headers="stat_1">19 (49%)</td>
<td class="gt_row gt_center" headers="stat_2">23 (43%)</td>
<td class="gt_row gt_center" headers="stat_3">72 (57%)</td>
<td class="gt_row gt_center" headers="stat_4">13 (48%)</td>
<td class="gt_row gt_center" headers="stat_5">3 (38%)</td>
<td class="gt_row gt_center" headers="p.value">0.4</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label" style="font-weight: bold">CSF Aβ42/40 ratio pg/ml, Mean (SD)</td>
<td class="gt_row gt_center" headers="n">229</td>
<td class="gt_row gt_center" headers="stat_0">0.076 (0.020)</td>
<td class="gt_row gt_center" headers="stat_1">0.078 (0.022)</td>
<td class="gt_row gt_center" headers="stat_2">0.075 (0.021)</td>
<td class="gt_row gt_center" headers="stat_3">0.075 (0.019)</td>
<td class="gt_row gt_center" headers="stat_4">0.077 (0.021)</td>
<td class="gt_row gt_center" headers="stat_5">0.077 (0.022)</td>
<td class="gt_row gt_center" headers="p.value">&gt;0.9</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label" style="font-weight: bold">Baseline Hippocampal volume (cm3), Mean (SD)</td>
<td class="gt_row gt_center" headers="n">197</td>
<td class="gt_row gt_center" headers="stat_0">5.35 (0.52)</td>
<td class="gt_row gt_center" headers="stat_1">5.31 (0.49)</td>
<td class="gt_row gt_center" headers="stat_2">5.25 (0.48)</td>
<td class="gt_row gt_center" headers="stat_3">5.39 (0.54)</td>
<td class="gt_row gt_center" headers="stat_4">5.41 (0.55)</td>
<td class="gt_row gt_center" headers="stat_5">5.22 (0.47)</td>
<td class="gt_row gt_center" headers="p.value">0.7</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label" style="font-weight: bold">Follow-up Hippocampal volume (cm3), Mean (SD)</td>
<td class="gt_row gt_center" headers="n">197</td>
<td class="gt_row gt_center" headers="stat_0">5.23 (0.53)</td>
<td class="gt_row gt_center" headers="stat_1">5.21 (0.50)</td>
<td class="gt_row gt_center" headers="stat_2">5.14 (0.50)</td>
<td class="gt_row gt_center" headers="stat_3">5.28 (0.56)</td>
<td class="gt_row gt_center" headers="stat_4">5.25 (0.56)</td>
<td class="gt_row gt_center" headers="stat_5">5.09 (0.40)</td>
<td class="gt_row gt_center" headers="p.value">0.7</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label" style="font-weight: bold">Baseline composite attention score, Mean (SD)</td>
<td class="gt_row gt_center" headers="n">187</td>
<td class="gt_row gt_center" headers="stat_0">-0.17 (0.77)</td>
<td class="gt_row gt_center" headers="stat_1">-0.03 (0.66)</td>
<td class="gt_row gt_center" headers="stat_2">-0.18 (0.76)</td>
<td class="gt_row gt_center" headers="stat_3">-0.27 (0.82)</td>
<td class="gt_row gt_center" headers="stat_4">0.12 (0.67)</td>
<td class="gt_row gt_center" headers="stat_5">-0.25 (0.73)</td>
<td class="gt_row gt_center" headers="p.value">0.4</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label" style="font-weight: bold">Follow-up composite attention score, Mean (SD)</td>
<td class="gt_row gt_center" headers="n">187</td>
<td class="gt_row gt_center" headers="stat_0">-0.05 (0.76)</td>
<td class="gt_row gt_center" headers="stat_1">0.13 (0.70)</td>
<td class="gt_row gt_center" headers="stat_2">-0.08 (0.79)</td>
<td class="gt_row gt_center" headers="stat_3">-0.11 (0.69)</td>
<td class="gt_row gt_center" headers="stat_4">0.12 (0.81)</td>
<td class="gt_row gt_center" headers="stat_5">-0.55 (1.37)</td>
<td class="gt_row gt_center" headers="p.value">0.4</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label" style="font-weight: bold">Baseline composite executive score, Mean (SD)</td>
<td class="gt_row gt_center" headers="n">183</td>
<td class="gt_row gt_center" headers="stat_0">-0.20 (0.82)</td>
<td class="gt_row gt_center" headers="stat_1">-0.01 (0.69)</td>
<td class="gt_row gt_center" headers="stat_2">-0.08 (0.70)</td>
<td class="gt_row gt_center" headers="stat_3">-0.34 (0.87)</td>
<td class="gt_row gt_center" headers="stat_4">0.04 (0.80)</td>
<td class="gt_row gt_center" headers="stat_5">-0.74 (1.12)</td>
<td class="gt_row gt_center" headers="p.value">0.10</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label" style="font-weight: bold">Follow-up composite executive score, Mean (SD)</td>
<td class="gt_row gt_center" headers="n">183</td>
<td class="gt_row gt_center" headers="stat_0">-0.21 (0.87)</td>
<td class="gt_row gt_center" headers="stat_1">0.01 (0.83)</td>
<td class="gt_row gt_center" headers="stat_2">-0.06 (0.79)</td>
<td class="gt_row gt_center" headers="stat_3">-0.38 (0.82)</td>
<td class="gt_row gt_center" headers="stat_4">0.10 (0.97)</td>
<td class="gt_row gt_center" headers="stat_5">-0.79 (1.28)</td>
<td class="gt_row gt_center" headers="p.value">0.020</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label" style="font-weight: bold">Baseline composite memory score, Mean (SD)</td>
<td class="gt_row gt_center" headers="n">182</td>
<td class="gt_row gt_center" headers="stat_0">0.11 (0.65)</td>
<td class="gt_row gt_center" headers="stat_1">0.24 (0.72)</td>
<td class="gt_row gt_center" headers="stat_2">0.03 (0.66)</td>
<td class="gt_row gt_center" headers="stat_3">0.07 (0.60)</td>
<td class="gt_row gt_center" headers="stat_4">0.21 (0.80)</td>
<td class="gt_row gt_center" headers="stat_5">0.07 (0.62)</td>
<td class="gt_row gt_center" headers="p.value">0.3</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label" style="font-weight: bold">Follow-up composite memory score, Mean (SD)</td>
<td class="gt_row gt_center" headers="n">182</td>
<td class="gt_row gt_center" headers="stat_0">0.13 (0.64)</td>
<td class="gt_row gt_center" headers="stat_1">0.19 (0.72)</td>
<td class="gt_row gt_center" headers="stat_2">0.11 (0.63)</td>
<td class="gt_row gt_center" headers="stat_3">0.08 (0.59)</td>
<td class="gt_row gt_center" headers="stat_4">0.32 (0.74)</td>
<td class="gt_row gt_center" headers="stat_5">0.03 (0.65)</td>
<td class="gt_row gt_center" headers="p.value">0.4</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label" style="font-weight: bold">Baseline composite language score, Mean (SD)</td>
<td class="gt_row gt_center" headers="n">200</td>
<td class="gt_row gt_center" headers="stat_0">0.05 (1.06)</td>
<td class="gt_row gt_center" headers="stat_1">0.16 (1.17)</td>
<td class="gt_row gt_center" headers="stat_2">-0.15 (0.95)</td>
<td class="gt_row gt_center" headers="stat_3">0.04 (1.05)</td>
<td class="gt_row gt_center" headers="stat_4">0.27 (1.05)</td>
<td class="gt_row gt_center" headers="stat_5">0.18 (1.29)</td>
<td class="gt_row gt_center" headers="p.value">0.6</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label" style="font-weight: bold">Follow-up composite language score, Mean (SD)</td>
<td class="gt_row gt_center" headers="n">200</td>
<td class="gt_row gt_center" headers="stat_0">-0.03 (1.03)</td>
<td class="gt_row gt_center" headers="stat_1">0.10 (1.01)</td>
<td class="gt_row gt_center" headers="stat_2">-0.24 (0.89)</td>
<td class="gt_row gt_center" headers="stat_3">-0.04 (1.05)</td>
<td class="gt_row gt_center" headers="stat_4">0.34 (1.02)</td>
<td class="gt_row gt_center" headers="stat_5">-0.51 (1.28)</td>
<td class="gt_row gt_center" headers="p.value">0.14</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label" style="font-weight: bold">Baseline composite visual score, Mean (SD)</td>
<td class="gt_row gt_center" headers="n">187</td>
<td class="gt_row gt_center" headers="stat_0">-0.25 (0.83)</td>
<td class="gt_row gt_center" headers="stat_1">-0.07 (0.86)</td>
<td class="gt_row gt_center" headers="stat_2">-0.21 (0.69)</td>
<td class="gt_row gt_center" headers="stat_3">-0.38 (0.84)</td>
<td class="gt_row gt_center" headers="stat_4">-0.01 (0.87)</td>
<td class="gt_row gt_center" headers="stat_5">-0.33 (0.97)</td>
<td class="gt_row gt_center" headers="p.value">0.3</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label" style="font-weight: bold">Follow-up composite visual score, Mean (SD)</td>
<td class="gt_row gt_center" headers="n">187</td>
<td class="gt_row gt_center" headers="stat_0">-0.23 (0.78)</td>
<td class="gt_row gt_center" headers="stat_1">-0.07 (0.65)</td>
<td class="gt_row gt_center" headers="stat_2">-0.21 (0.70)</td>
<td class="gt_row gt_center" headers="stat_3">-0.28 (0.82)</td>
<td class="gt_row gt_center" headers="stat_4">-0.23 (0.85)</td>
<td class="gt_row gt_center" headers="stat_5">-0.31 (1.14)</td>
<td class="gt_row gt_center" headers="p.value">0.6</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label" style="font-weight: bold">Baseline Total Intracranial Volume (cm3), Mean (SD)</td>
<td class="gt_row gt_center" headers="n">197</td>
<td class="gt_row gt_center" headers="stat_0">1,253 (93)</td>
<td class="gt_row gt_center" headers="stat_1">1,257 (78)</td>
<td class="gt_row gt_center" headers="stat_2">1,235 (87)</td>
<td class="gt_row gt_center" headers="stat_3">1,259 (98)</td>
<td class="gt_row gt_center" headers="stat_4">1,251 (108)</td>
<td class="gt_row gt_center" headers="stat_5">1,275 (64)</td>
<td class="gt_row gt_center" headers="p.value">0.6</td>
</tr>
</tbody><tfoot class="gt_footnotes">
<tr class="odd">
<td colspan="9" class="gt_footnote"><span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height:0;"><sup>1</sup></span> Mean (SD); n (%)</td>
</tr>
<tr class="even">
<td colspan="9" class="gt_footnote"><span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height:0;"><sup>2</sup></span> Kruskal-Wallis rank sum test; Fisher’s exact test</td>
</tr>
</tfoot>

</table>

</div>
</div>
</div>
<p>As shown in Table 1, 39 (15.3%) participants had no previous childbirths, 54 (21.2%) had one previous childbirth, 126 (49.6%) had 2, 27 (10.6%) had 3, and only 8 (3%) had 4 or more. Age, education and follow-up executive composite score significantly differed depending on parity. No differences were observed among groups for Aβ, APOE-ε4 carrier status, or HV.</p>
<p>Carriers of APOE-E4 had significantly higher CSF Aβ42/40 load compared with non carriers (t = 5, df = 191, p &lt; 0.001).</p>
</section>
<section id="interplay-between-parity-apoe-e4-and-hv" class="level2" data-number="4.2">
<h2 data-number="4.2" class="anchored" data-anchor-id="interplay-between-parity-apoe-e4-and-hv"><span class="header-section-number">4.2</span> Interplay between parity, APOE-E4 and HV</h2>
<p>Parity, APOE-ε4, and time showed a significant three-way interaction for total HV (β = -0.019, SE = 0.006, p = 0.001), indicating that higher parity was associated with steeper decline in HV among APOE-ε4 carriers. The triple interaction effect was maintained when adjusting analyses by Aβ accumulation (β = -0.026, SE = 0.008, p = 0.001). Furthermore, the effect was maintained, although attenuated, when reproducing the analysis including only participants with negative Aβ status (CSF Aβ&lt;0.071 pg/ml, n = 128) (β = -0.03, SE = 0.012, p = 0.016).</p>
<p>Stratifying by APOE-ε4 status yielded a significant interaction between parity and time in carriers only (β = -0.012, SE = 0.004, p = 0.001). Again, results were maintained when adjusting the model by Aβ accumulation (β = 0.595, SE = 1.199, p = 0.621).</p>
<p>Reproducing the analysis using parity as a dichotomous variable reduced the 3-way interaction estimate and significance of the effect (β = -0.031, SE = 0.016, p = 0.051).</p>
<p>The 3-way interaction between APOE4 parity and time was not associated with GMV in the calcarine cortical area.</p>
</section>
<section id="sensitivity-analysis-with-hormonal-variables" class="level2" data-number="4.3">
<h2 data-number="4.3" class="anchored" data-anchor-id="sensitivity-analysis-with-hormonal-variables"><span class="header-section-number">4.3</span> Sensitivity analysis with hormonal variables</h2>
<p>Adding other reproductive variables to our main analysis did not have an effect on the 3-way estimate.</p>
</section>
<section id="cognitive-trajectory" class="level2" data-number="4.4">
<h2 data-number="4.4" class="anchored" data-anchor-id="cognitive-trajectory"><span class="header-section-number">4.4</span> Cognitive trajectory</h2>
<p>The interaction between parity, APOE4 and time yielded a weak trend which was not present after FDR correction. (not same direction so not sure if worth mentioning)</p>
<ul>
<li>talk about lower order interaction if present? they are a bit weird direction!</li>
</ul>
</section>
<section id="exploratory-analysis-of-hippocampal-subfields" class="level2" data-number="4.5">
<h2 data-number="4.5" class="anchored" data-anchor-id="exploratory-analysis-of-hippocampal-subfields"><span class="header-section-number">4.5</span> Exploratory analysis of hippocampal subfields</h2>
<p>The 3-way interaction between parity, APOE4 and time was significant in the CA1 (β= -0.036, SE = 0.013, p= 0.006), CA2 (β= -0.068, SE = 0.029, p= 0.023) and in the DG (β= -0.033, SE = 0.015, p= 0.029). Similarly, parity was associated with steeper HV decline in APOE-E4 carriers.</p>
<p>Adjusting by AB….</p>
</section>
<section id="exploratory-analysis-of-inflammatory-biomarkers" class="level2" data-number="4.6">
<h2 data-number="4.6" class="anchored" data-anchor-id="exploratory-analysis-of-inflammatory-biomarkers"><span class="header-section-number">4.6</span> Exploratory analysis of inflammatory biomarkers</h2>
<p>There was no association with any of the biomarkers assessed.</p>
</section>
</section>
<section id="discussion" class="level1" data-number="5">
<h1 data-number="5"><span class="header-section-number">5</span> Discussion</h1>
<p>Our study aimed to investigate the impact of parity on hippocampal subfield volumes and relation to the APOE4 allele in post-menopausal women at risk of AD. Our results suggest that parity and AD specific genotype interact in women to impact hippocampal volume loss. In our sample, multiparous APOE4 non-carriers showed less hippocampus volume decline, whilst multiparous APOE4 carriers showed worse cognitive decline. These rseults show evidence of an interaction between parity and AD related genetic risk in women, which is associated in volume loss in a brain region which is a prime marker of AD. Essentially, this evidence strongly suggests there is a clear biological component to the interaction between parity and AD risk in women.</p>
<p>When using the categorical grouping for parity, results showed significantly bigger effect in multiparous women. This could indicate that the cumulative effects of parity are only significantly interact with genetic risk after having had more than one child. However, it is important to note that the multiparous groups had a fairly larger number of observations than the nulliparous or primiparous group, therefore lack of significance might indicate a power issue when performing the gorup interaction.</p>
<p>The AFLFA+ cohort is an ideal cohort to test this interaction, given the participants are enriched in genetic AD risk, As such, about 40% of our cohort is a carrier of the APOE4 allele, meaning group interactions are very balanced statistically.</p>
<p>Our findings reproduce the ones found by Lee and colleagues. Similarly, APOE4 carriers show detrimental effects of parity on hippocampal volume trajectory. Although we did not replicate their findings on memory, this could be due to the relative short amount of time between both visits, on 3 years in average. However, another hypothesis could be that there are intricate social factors at play in humans revolving around parity, which could be influencing cognitive resilience. A previous study done with the UK biobank also finds no effect of the interaction between APOE4 and parity on cognitive score (lindseth et al 2023), although their battery of test is limited.</p>
<p>Our post-hoc analysis of hippocampal subfields shows the most affected regions are the CA1, CA2, subiculum and enthorinal cortex, and a trend in the dentate gyrus. Given Bee’s results, one could expect a stornger effect in the dentate gyrus, given most of the effect of parity in neurogenesis would reside in this subfield. The strong effect on parts of the cornu ammonis also reveals ageing mechanism.</p>
<p>Some limitations should be considered. Firstly, the number of follow-up visits is insufficient to observe nonlinear evolution, therefore limiting our understanding of possible mechanisms underlying AD pathogenesis. Future work should include additional time points to better understand the trajectory of these results. Secondly, although our sample’s excellent health record makes it ideal to identify subtle changes in the early AD continuum, the vast majority of ALFA+ study participants are white Caucasians, resulting in an ethnically homogeneous sample primarily from middle to upper-middle class backgrounds. Future work should reproduce these analyses in a more diverse cohort to understand the impact of ethnicity and social background on the present results. Lastly, the absence of direct hormonal measurements limits our ability to draw definitive conclusions about the role of sex steroid hormones in the observed associations. By relying on proxy measures, we can only estimate hormone-related effects, potentially leading to an underestimation of their actual influence.</p>
</section>
<section id="figures" class="level1" data-number="6">
<h1 data-number="6"><span class="header-section-number">6</span> Figures</h1>
<section id="figure-1" class="level2" data-number="6.1">
<h2 data-number="6.1" class="anchored" data-anchor-id="figure-1"><span class="header-section-number">6.1</span> Figure 1</h2>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="paper_ashs_pregnancy_APOE_files/figure-html/unnamed-chunk-13-1.png" class="img-fluid figure-img" style="width:100.0%"></p>
</figure>
</div>
</div>
</div>
</section>
<section id="figure-2" class="level2" data-number="6.2">
<h2 data-number="6.2" class="anchored" data-anchor-id="figure-2"><span class="header-section-number">6.2</span> Figure 2</h2>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="paper_ashs_pregnancy_APOE_files/figure-html/unnamed-chunk-16-1.png" class="img-fluid figure-img" style="width:100.0%"></p>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="appendix" class="level1" data-number="7">
<h1 data-number="7"><span class="header-section-number">7</span> Appendix</h1>
<section id="mri-data-acquisition" class="level2" data-number="7.1">
<h2 data-number="7.1" class="anchored" data-anchor-id="mri-data-acquisition"><span class="header-section-number">7.1</span> MRI data acquisition</h2>
<p>MRI scans were acquired on a in-house 3.0 T scanner (GE Discovery MR750 W 3 T) using the same protocol for all participants, which included one T1- and three T2- weighted sequences. The 3D-T1w sequence was designed with an isotropic voxel size of1mm3 and a matrix size of256x256x160 (TR/TE/TI = 8.0/3.7/450 ms, NSA = 1, flip angle = 8°). Three 3D-T2w sequences, with a voxel size of1 mm× 1 mm× 3 mm, were also used: fluid attenuation inversion recovery (FLAIR: TR/TE/ TI = 11,000/90/2600 ms, flip angle = 160°), fast spin echo (FSE: TR/TE = 5000/85 ms, flip angle = 110°) and, gradient echo (GRE: TR/TE = 1300/23 ms, flip angle = 15°). All scans were visually assessed to verify their quality and to detect incidental findings by a trained neuroradiologist and have been reported elsewhere (Brugulat-Serrat et al.&nbsp;2017). At visit 1, ten participants were excluded due to the presence of a meningioma, as well as 37 participants due to susceptibility, motion artefacts or segmentation problems, resulting in a total of 561 images available for subsequent analysis. At Visit 2, … The medial temporal lobe atrophy was assessed by Medial Temporal Atrophy scale <span class="citation" data-cites="scheltens1992a">[<a href="#ref-scheltens1992a" role="doc-biblioref">30</a>]</span>.</p>
</section>
</section>
<section id="miscellaneous" class="level1" data-number="8">
<h1 data-number="8"><span class="header-section-number">8</span> Miscellaneous</h1>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a>preds_ashs <span class="ot">&lt;-</span> <span class="fu">ggpredict</span>(mod_ashs_APOE, <span class="at">terms =</span> <span class="fu">c</span>(<span class="st">"diff_mri"</span>, <span class="st">"kids_total"</span>, <span class="st">"APOE_binary"</span>))</span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a>preds_ashs_A0<span class="ot">&lt;-</span><span class="fu">as.data.frame</span>(preds_ashs)<span class="sc">%&gt;%</span></span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">rename</span>(<span class="at">kids_total_factor =</span> group,</span>
<span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a>         <span class="at">APOE_binary=</span>facet)<span class="sc">%&gt;%</span></span>
<span id="cb1-6"><a href="#cb1-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(APOE_binary<span class="sc">==</span><span class="dv">0</span>)<span class="sc">%&gt;%</span></span>
<span id="cb1-7"><a href="#cb1-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">APOE_binary =</span> <span class="st">"Non-Carrier"</span>,</span>
<span id="cb1-8"><a href="#cb1-8" aria-hidden="true" tabindex="-1"></a>         <span class="at">kids_total_factor=</span> <span class="fu">ifelse</span>(<span class="fu">as.character</span>(kids_total_factor)<span class="sc">==</span><span class="dv">4</span>, <span class="st">"4+"</span>, <span class="fu">as.character</span>(kids_total_factor)))</span>
<span id="cb1-9"><a href="#cb1-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-10"><a href="#cb1-10" aria-hidden="true" tabindex="-1"></a>preds_ashs_A1<span class="ot">&lt;-</span><span class="fu">as.data.frame</span>(preds_ashs)<span class="sc">%&gt;%</span></span>
<span id="cb1-11"><a href="#cb1-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">rename</span>(<span class="at">kids_total_factor =</span> group,</span>
<span id="cb1-12"><a href="#cb1-12" aria-hidden="true" tabindex="-1"></a>         <span class="at">APOE_binary=</span>facet)<span class="sc">%&gt;%</span></span>
<span id="cb1-13"><a href="#cb1-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(APOE_binary<span class="sc">==</span><span class="dv">1</span>)<span class="sc">%&gt;%</span></span>
<span id="cb1-14"><a href="#cb1-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">APOE_binary =</span> <span class="st">"Carrier"</span>,</span>
<span id="cb1-15"><a href="#cb1-15" aria-hidden="true" tabindex="-1"></a>         <span class="at">kids_total_factor=</span> <span class="fu">ifelse</span>(<span class="fu">as.character</span>(kids_total_factor)<span class="sc">==</span><span class="dv">4</span>, <span class="st">"4+"</span>, <span class="fu">as.character</span>(kids_total_factor)))</span>
<span id="cb1-16"><a href="#cb1-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-17"><a href="#cb1-17" aria-hidden="true" tabindex="-1"></a><span class="co"># Create a combined factor for color mapping</span></span>
<span id="cb1-18"><a href="#cb1-18" aria-hidden="true" tabindex="-1"></a>data_plot <span class="ot">&lt;-</span> data_ashs <span class="sc">%&gt;%</span></span>
<span id="cb1-19"><a href="#cb1-19" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(sex <span class="sc">==</span> <span class="dv">2</span>, <span class="sc">!</span><span class="fu">is.na</span>(APOE_binary)) <span class="sc">%&gt;%</span></span>
<span id="cb1-20"><a href="#cb1-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">kids_total_factor =</span> <span class="fu">ifelse</span>(kids_total<span class="sc">==</span><span class="dv">4</span>, <span class="st">"4+"</span>, kids_total),</span>
<span id="cb1-21"><a href="#cb1-21" aria-hidden="true" tabindex="-1"></a>         <span class="at">APOE_binary =</span> <span class="fu">factor</span>(APOE_binary, <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"0"</span>, <span class="st">"1"</span>), <span class="at">labels =</span> <span class="fu">c</span>(<span class="st">"Non-Carrier"</span>, <span class="st">"Carrier"</span>)),</span>
<span id="cb1-22"><a href="#cb1-22" aria-hidden="true" tabindex="-1"></a>         <span class="at">color_group =</span> <span class="fu">paste</span>(APOE_binary, kids_total_factor, <span class="at">sep =</span> <span class="st">"_"</span>))</span>
<span id="cb1-23"><a href="#cb1-23" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-24"><a href="#cb1-24" aria-hidden="true" tabindex="-1"></a>preds_ashs_A0 <span class="ot">&lt;-</span> preds_ashs_A0 <span class="sc">%&gt;%</span></span>
<span id="cb1-25"><a href="#cb1-25" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">color_group =</span> <span class="fu">paste</span>(<span class="st">"Non-Carrier"</span>, kids_total_factor, <span class="at">sep =</span> <span class="st">"_"</span>))</span>
<span id="cb1-26"><a href="#cb1-26" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-27"><a href="#cb1-27" aria-hidden="true" tabindex="-1"></a>preds_ashs_A1 <span class="ot">&lt;-</span> preds_ashs_A1 <span class="sc">%&gt;%</span></span>
<span id="cb1-28"><a href="#cb1-28" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">color_group =</span> <span class="fu">paste</span>(<span class="st">"Carrier"</span>, kids_total_factor, <span class="at">sep =</span> <span class="st">"_"</span>))</span>
<span id="cb1-29"><a href="#cb1-29" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-30"><a href="#cb1-30" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-31"><a href="#cb1-31" aria-hidden="true" tabindex="-1"></a><span class="co"># Define color palettes</span></span>
<span id="cb1-32"><a href="#cb1-32" aria-hidden="true" tabindex="-1"></a>color_values <span class="ot">&lt;-</span> <span class="fu">c</span>(</span>
<span id="cb1-33"><a href="#cb1-33" aria-hidden="true" tabindex="-1"></a>  <span class="st">"Non-Carrier_0"</span> <span class="ot">=</span> <span class="st">"#4DD0E1"</span>, <span class="st">"Non-Carrier_1"</span> <span class="ot">=</span> <span class="st">"#1CA8B7"</span>, <span class="st">"Non-Carrier_2"</span> <span class="ot">=</span> <span class="st">"#1E8895"</span>, <span class="st">"Non-Carrier_3"</span> <span class="ot">=</span> <span class="st">"#0D707B"</span>, <span class="st">"Non-Carrier_4+"</span> <span class="ot">=</span> <span class="st">"#0F565E"</span>,</span>
<span id="cb1-34"><a href="#cb1-34" aria-hidden="true" tabindex="-1"></a>  <span class="st">"Carrier_0"</span> <span class="ot">=</span> <span class="st">"#FF8A65"</span>, <span class="st">"Carrier_1"</span> <span class="ot">=</span> <span class="st">"#FF7043"</span>, <span class="st">"Carrier_2"</span> <span class="ot">=</span> <span class="st">"#FF5722"</span>, <span class="st">"Carrier_3"</span> <span class="ot">=</span> <span class="st">"#F4511E"</span>, <span class="st">"Carrier_4+"</span> <span class="ot">=</span> <span class="st">"#BF360C"</span></span>
<span id="cb1-35"><a href="#cb1-35" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb1-36"><a href="#cb1-36" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-37"><a href="#cb1-37" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(data_plot, <span class="fu">aes</span>(<span class="at">x =</span> diff_mri, <span class="at">y =</span> vol_HIPP, <span class="at">color =</span> color_group)) <span class="sc">+</span></span>
<span id="cb1-38"><a href="#cb1-38" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>(<span class="at">alpha =</span> <span class="fl">0.3</span>, <span class="at">size=</span><span class="dv">2</span>) <span class="sc">+</span></span>
<span id="cb1-39"><a href="#cb1-39" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>(<span class="at">data =</span> preds_ashs_A0, <span class="fu">aes</span>(<span class="at">x=</span>x, <span class="at">y=</span>predicted, <span class="at">group=</span>kids_total_factor, <span class="at">color=</span>color_group), <span class="at">size=</span><span class="fl">1.3</span>) <span class="sc">+</span></span>
<span id="cb1-40"><a href="#cb1-40" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>(<span class="at">data =</span> preds_ashs_A1, <span class="fu">aes</span>(<span class="at">x=</span>x, <span class="at">y=</span>predicted, <span class="at">group=</span>kids_total_factor, <span class="at">color=</span>color_group), <span class="at">size=</span><span class="fl">1.3</span>) <span class="sc">+</span></span>
<span id="cb1-41"><a href="#cb1-41" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_ribbon</span>(<span class="at">data =</span> preds_ashs_A0, <span class="fu">aes</span>(<span class="at">x=</span>x, <span class="at">ymin=</span>conf.low, <span class="at">ymax=</span>conf.high, </span>
<span id="cb1-42"><a href="#cb1-42" aria-hidden="true" tabindex="-1"></a>                                         <span class="at">group=</span>kids_total_factor, <span class="at">fill=</span>color_group), </span>
<span id="cb1-43"><a href="#cb1-43" aria-hidden="true" tabindex="-1"></a>              <span class="at">inherit.aes =</span> F, <span class="at">alpha=</span>.<span class="dv">15</span>)<span class="sc">+</span></span>
<span id="cb1-44"><a href="#cb1-44" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_ribbon</span>(<span class="at">data =</span> preds_ashs_A1, <span class="fu">aes</span>(<span class="at">x=</span>x, <span class="at">ymin=</span>conf.low, <span class="at">ymax=</span>conf.high, </span>
<span id="cb1-45"><a href="#cb1-45" aria-hidden="true" tabindex="-1"></a>                                         <span class="at">group=</span>kids_total_factor, <span class="at">fill=</span>color_group), </span>
<span id="cb1-46"><a href="#cb1-46" aria-hidden="true" tabindex="-1"></a>              <span class="at">inherit.aes =</span> F, <span class="at">alpha=</span>.<span class="dv">15</span>)<span class="sc">+</span></span>
<span id="cb1-47"><a href="#cb1-47" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_wrap</span>(<span class="sc">~</span>APOE_binary) <span class="sc">+</span></span>
<span id="cb1-48"><a href="#cb1-48" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_manual</span>(<span class="at">values =</span> color_values, <span class="at">labels =</span> <span class="fu">c</span>(<span class="st">"0"</span>, <span class="st">"1"</span>, <span class="st">"2"</span>, <span class="st">"3"</span>, <span class="st">"4+"</span>, <span class="st">"0"</span>, <span class="st">"1"</span>, <span class="st">"2"</span>, <span class="st">"3"</span>, <span class="st">"4+"</span>)) <span class="sc">+</span></span>
<span id="cb1-49"><a href="#cb1-49" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> color_values, <span class="at">labels =</span> <span class="fu">c</span>(<span class="st">"0"</span>, <span class="st">"1"</span>, <span class="st">"2"</span>, <span class="st">"3"</span>, <span class="st">"4+"</span>, <span class="st">"0"</span>, <span class="st">"1"</span>, <span class="st">"2"</span>, <span class="st">"3"</span>, <span class="st">"4+"</span>)) <span class="sc">+</span></span>
<span id="cb1-50"><a href="#cb1-50" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="st">"Time difference (days)"</span>, </span>
<span id="cb1-51"><a href="#cb1-51" aria-hidden="true" tabindex="-1"></a>       <span class="at">y =</span> <span class="st">"PACC Z-score"</span>, </span>
<span id="cb1-52"><a href="#cb1-52" aria-hidden="true" tabindex="-1"></a>       <span class="at">color =</span> <span class="st">"Number of childbirths"</span>,</span>
<span id="cb1-53"><a href="#cb1-53" aria-hidden="true" tabindex="-1"></a>       <span class="at">fill =</span> <span class="st">"Number of childbirths"</span>)<span class="sc">+</span></span>
<span id="cb1-54"><a href="#cb1-54" aria-hidden="true" tabindex="-1"></a>  th</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>

</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography" id="quarto-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body" data-entry-spacing="0" role="list">
<div id="ref-riedel2016" class="csl-entry" role="listitem">
<div class="csl-left-margin">[1] </div><div class="csl-right-inline">Riedel BC, Thompson PM, Brinton RD. Age, APOE and sex: Triad of risk of Alzheimer<span>’</span>s disease. The Journal of Steroid Biochemistry and Molecular Biology 2016;160:134–47. <a href="https://doi.org/10.1016/j.jsbmb.2016.03.012">https://doi.org/10.1016/j.jsbmb.2016.03.012</a>.</div>
</div>
<div id="ref-corder1993" class="csl-entry" role="listitem">
<div class="csl-left-margin">[2] </div><div class="csl-right-inline">Corder EH, Saunders AM, Strittmatter WJ, Schmechel DE, Gaskell PC, Small GW, et al. Gene Dose of Apolipoprotein E Type 4 Allele and the Risk of Alzheimer’s Disease in Late Onset Families. Science 1993;261:921–3. <a href="https://doi.org/10.1126/science.8346443">https://doi.org/10.1126/science.8346443</a>.</div>
</div>
<div id="ref-beam2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">[3] </div><div class="csl-right-inline">Beam CR, Kaneshiro C, Jang JY, Reynolds CA, Pedersen NL, Gatz M. Differences Between Women and Men in Incidence Rates of Dementia and Alzheimer<span>’</span>s Disease. Journal of Alzheimer<span>’</span>s Disease 2018;64:1077–83. <a href="https://doi.org/10.3233/jad-180141">https://doi.org/10.3233/jad-180141</a>.</div>
</div>
<div id="ref-sexdiff2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">[4] </div><div class="csl-right-inline">Ferretti MT, Iulita MF, Cavedo E, Chiesa PA, Schumacher Dimech A, Santuccione Chadha A, et al. Sex differences in Alzheimer disease <span></span> the gateway to precision medicine. Nature Reviews Neurology 2018;14:457–69. <a href="https://doi.org/10.1038/s41582-018-0032-9">https://doi.org/10.1038/s41582-018-0032-9</a>.</div>
</div>
<div id="ref-Altmann2014" class="csl-entry" role="listitem">
<div class="csl-left-margin">[5] </div><div class="csl-right-inline">Altmann A, Tian L, Henderson VW, Greicius MD. Sex modifies the <span><em>APOE</em></span><span>-</span>related risk of developing Alzheimer disease. Annals of Neurology 2014;75:563–73. <a href="https://doi.org/10.1002/ana.24135">https://doi.org/10.1002/ana.24135</a>.</div>
</div>
<div id="ref-Neu2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">[6] </div><div class="csl-right-inline">Neu SC, Pa J, Kukull W, Beekly D, Kuzma A, Gangadharan P, et al. Apolipoprotein E Genotype and Sex Risk Factors for Alzheimer Disease. JAMA Neurology 2017;74:1178. <a href="https://doi.org/10.1001/jamaneurol.2017.2188">https://doi.org/10.1001/jamaneurol.2017.2188</a>.</div>
</div>
<div id="ref-apostolova2006" class="csl-entry" role="listitem">
<div class="csl-left-margin">[7] </div><div class="csl-right-inline">Apostolova LG, Dutton RA, Dinov ID, Hayashi KM, Toga AW, Cummings JL, et al. Conversion of Mild Cognitive Impairment to Alzheimer Disease Predicted by Hippocampal Atrophy Maps. Archives of Neurology 2006;63:693. <a href="https://doi.org/10.1001/archneur.63.5.693">https://doi.org/10.1001/archneur.63.5.693</a>.</div>
</div>
<div id="ref-pievani2011" class="csl-entry" role="listitem">
<div class="csl-left-margin">[8] </div><div class="csl-right-inline">Pievani M, Galluzzi S, Thompson PM, Rasser PE, Bonetti M, Frisoni GB. APOE4 is associated with greater atrophy of the hippocampal formation in Alzheimer’s disease. NeuroImage 2011;55:909–19. <a href="https://doi.org/10.1016/j.neuroimage.2010.12.081">https://doi.org/10.1016/j.neuroimage.2010.12.081</a>.</div>
</div>
<div id="ref-puri2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">[9] </div><div class="csl-right-inline">Puri TA, Richard JE, Galea LAM. Beyond sex differences: short- and long-term effects of pregnancy on the brain. Trends in Neurosciences 2023;46:459–71. <a href="https://doi.org/10.1016/j.tins.2023.03.010">https://doi.org/10.1016/j.tins.2023.03.010</a>.</div>
</div>
<div id="ref-pawluski2007" class="csl-entry" role="listitem">
<div class="csl-left-margin">[10] </div><div class="csl-right-inline">Pawluski JL, Galea LAM. Reproductive experience alters hippocampal neurogenesis during the postpartum period in the dam. Neuroscience 2007;149:53–67. <a href="https://doi.org/10.1016/j.neuroscience.2007.07.031">https://doi.org/10.1016/j.neuroscience.2007.07.031</a>.</div>
</div>
<div id="ref-eid2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">[11] </div><div class="csl-right-inline">Eid RS, Chaiton JA, Lieblich SE, Bodnar TS, Weinberg J, Galea LAM. Early and late effects of maternal experience on hippocampal neurogenesis, microglia, and the circulating cytokine milieu. Neurobiology of Aging 2019;78:1–17. <a href="https://doi.org/10.1016/j.neurobiolaging.2019.01.021">https://doi.org/10.1016/j.neurobiolaging.2019.01.021</a>.</div>
</div>
<div id="ref-lee2024" class="csl-entry" role="listitem">
<div class="csl-left-margin">[12] </div><div class="csl-right-inline">Lee BH, Cevizci M, Lieblich SE, Ibrahim M, Wen Y, Eid RS, et al. Exploring the parity paradox: Differential effects on neuroplasticity and inflammation by APOEe4 genotype at middle age. Brain, Behavior, and Immunity 2024;120:54–70. <a href="https://doi.org/10.1016/j.bbi.2024.05.019">https://doi.org/10.1016/j.bbi.2024.05.019</a>.</div>
</div>
<div id="ref-Hoekzema2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">[13] </div><div class="csl-right-inline">Hoekzema E, Barba-Müller E, Pozzobon C, Picado M, Lucco F, García-García D, et al. Pregnancy leads to long-lasting changes in human brain structure. Nature Neuroscience 2016;20:287–96. <a href="https://doi.org/10.1038/nn.4458">https://doi.org/10.1038/nn.4458</a>.</div>
</div>
<div id="ref-martínez-garcía2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">[14] </div><div class="csl-right-inline">Martínez-García M, Paternina-Die M, Barba-Müller E, Martín de Blas D, Beumala L, Cortizo R, et al. Do Pregnancy-Induced Brain Changes Reverse? The Brain of a Mother Six Years after Parturition. Brain Sciences 2021;11:168. <a href="https://doi.org/10.3390/brainsci11020168">https://doi.org/10.3390/brainsci11020168</a>.</div>
</div>
<div id="ref-de2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">[15] </div><div class="csl-right-inline">Lange A-MG de, Barth C, Kaufmann T, Anatürk M, Suri S, Ebmeier KP, et al. The maternal brain: Region<span>-</span>specific patterns of brain aging are traceable decades after childbirth. Human Brain Mapping 2020;41:4718–29. <a href="https://doi.org/10.1002/hbm.25152">https://doi.org/10.1002/hbm.25152</a>.</div>
</div>
<div id="ref-colucci2006" class="csl-entry" role="listitem">
<div class="csl-left-margin">[16] </div><div class="csl-right-inline">Colucci M, Cammarata S, Assini A, Croce R, Clerici F, Novello C, et al. The number of pregnancies is a risk factor for Alzheimer’s disease. European Journal of Neurology 2006;13:1374–7. <a href="https://doi.org/10.1111/j.1468-1331.2006.01520.x">https://doi.org/10.1111/j.1468-1331.2006.01520.x</a>.</div>
</div>
<div id="ref-Fox2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">[17] </div><div class="csl-right-inline">Fox M, Berzuini C, Knapp LA, Glynn LM. Women<span>’</span>s Pregnancy Life History and Alzheimer<span>’</span>s Risk: Can Immunoregulation Explain the Link? American Journal of Alzheimer’s Disease &amp; Other Dementias® 2018;33:516–26. <a href="https://doi.org/10.1177/1533317518786447">https://doi.org/10.1177/1533317518786447</a>.</div>
</div>
<div id="ref-Fox2013" class="csl-entry" role="listitem">
<div class="csl-left-margin">[18] </div><div class="csl-right-inline">Fox M, Berzuini C, Knapp LA. Cumulative estrogen exposure, number of menstrual cycles, and Alzheimer’s risk in a cohort of British women. Psychoneuroendocrinology 2013;38:2973–82. <a href="https://doi.org/10.1016/j.psyneuen.2013.08.005">https://doi.org/10.1016/j.psyneuen.2013.08.005</a>.</div>
</div>
<div id="ref-wolfova2024" class="csl-entry" role="listitem">
<div class="csl-left-margin">[19] </div><div class="csl-right-inline">Wolfova K, Hubbard RA, Brennan Kearns P, Chang VW, Crane P, LaCroix AZ, et al. Number of children and risk of dementia: a cohort study. Journal of Epidemiology and Community Health 2024;79:280–7. <a href="https://doi.org/10.1136/jech-2024-222717">https://doi.org/10.1136/jech-2024-222717</a>.</div>
</div>
<div id="ref-Beeri2009a" class="csl-entry" role="listitem">
<div class="csl-left-margin">[20] </div><div class="csl-right-inline">Beeri MS, Rapp M, Schmeidler J, Reichenberg A, Purohit DP, Perl DP, et al. Number of children is associated with neuropathology of Alzheimer’s disease in women. Neurobiology of Aging 2009;30:1184–91. <a href="https://doi.org/10.1016/j.neurobiolaging.2007.11.011">https://doi.org/10.1016/j.neurobiolaging.2007.11.011</a>.</div>
</div>
<div id="ref-molinuevo2016" class="csl-entry" role="listitem">
<div class="csl-left-margin">[21] </div><div class="csl-right-inline">Molinuevo JL, Gramunt N, Gispert JD, Fauria K, Esteller M, Minguillon C, et al. The ALFA project: A research platform to identify early pathophysiological features of Alzheimer’s disease. Alzheimer’s &amp; Dementia: Translational Research &amp; Clinical Interventions 2016;2:82–92. <a href="https://doi.org/10.1016/j.trci.2016.02.003">https://doi.org/10.1016/j.trci.2016.02.003</a>.</div>
</div>
<div id="ref-Milà-Alomà2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">[22] </div><div class="csl-right-inline">Milà-Alomà M, Salvadó G, Gispert JD, Vilor-Tejedor N, Grau-Rivera O, Sala-Vila A, et al. Amyloid beta, tau, synaptic, neurodegeneration, and glial biomarkers in the preclinical stage of the Alzheimer’s <span><em>continuum</em></span>. Alzheimer’s &amp; Dementia 2020;16:1358–71. <a href="https://doi.org/10.1002/alz.12131">https://doi.org/10.1002/alz.12131</a>.</div>
</div>
<div id="ref-Yushkevich2015" class="csl-entry" role="listitem">
<div class="csl-left-margin">[23] </div><div class="csl-right-inline">Yushkevich PA, Pluta JB, Wang H, Xie L, Ding S-L, Gertje EC, et al. Automated volumetry and regional thickness analysis of hippocampal subfields and medial temporal cortical structures in mild cognitive impairment. Human Brain Mapping 2014;36:258–87. <a href="https://doi.org/10.1002/hbm.22627">https://doi.org/10.1002/hbm.22627</a>.</div>
</div>
<div id="ref-smith2002" class="csl-entry" role="listitem">
<div class="csl-left-margin">[24] </div><div class="csl-right-inline">Smith SM. Fast robust automated brain extraction. Human Brain Mapping 2002;17:143–55. <a href="https://doi.org/10.1002/hbm.10062">https://doi.org/10.1002/hbm.10062</a>.</div>
</div>
<div id="ref-lenth2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">[25] </div><div class="csl-right-inline">Lenth RV, Piaskowski J. Emmeans: Estimated marginal means, aka least-squares means 2017. <a href="https://doi.org/10.32614/cran.package.emmeans">https://doi.org/10.32614/cran.package.emmeans</a>.</div>
</div>
<div id="ref-wickham2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">[26] </div><div class="csl-right-inline">Wickham H, Averick M, Bryan J, Chang W, McGowan L, François R, et al. Welcome to the tidyverse. Journal of Open Source Software 2019;4:1686. <a href="https://doi.org/10.21105/joss.01686">https://doi.org/10.21105/joss.01686</a>.</div>
</div>
<div id="ref-sjoberg2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">[27] </div><div class="csl-right-inline">Sjoberg Daniel,D, Whiting K, Curry M, Lavery Jessica,A, Larmarange J. Reproducible Summary Tables with the gtsummary Package. The R Journal 2021;13:570. <a href="https://doi.org/10.32614/rj-2021-053">https://doi.org/10.32614/rj-2021-053</a>.</div>
</div>
<div id="ref-lüdecke2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">[28] </div><div class="csl-right-inline">Lüdecke D. Ggeffects: Tidy data frames of marginal effects from regression models. Journal of Open Source Software 2018;3:772. <a href="https://doi.org/10.21105/joss.00772">https://doi.org/10.21105/joss.00772</a>.</div>
</div>
<div id="ref-bates2015" class="csl-entry" role="listitem">
<div class="csl-left-margin">[29] </div><div class="csl-right-inline">Bates D, Mächler M, Bolker B, Walker S. Fitting Linear Mixed-Effects Models Using<span><strong>lme4</strong></span>. Journal of Statistical Software 2015;67. <a href="https://doi.org/10.18637/jss.v067.i01">https://doi.org/10.18637/jss.v067.i01</a>.</div>
</div>
<div id="ref-scheltens1992a" class="csl-entry" role="listitem">
<div class="csl-left-margin">[30] </div><div class="csl-right-inline">Scheltens P, Weinstein HC, Leys D. Neuro-imaging in the diagnosis of Alzheimer’s disease I. Computer tomography and magnetic resonance imaging. Clinical Neurology and Neurosurgery 1992;94:277–89. <a href="https://doi.org/10.1016/0303-8467(92)90175-3">https://doi.org/10.1016/0303-8467(92)90175-3</a>.</div>
</div>
</div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>